
<!-- saved from url=(0056)https://evolution.gs.washington.edu/phylip/software.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta name="description" content="Phylogeny programs page describing all known software for inferring phylogenies (evolutionary trees)">
<meta name="keywords" content="phylogeny, phylogenies, evolutionary trees, evolution, cladistics, software">
<title>Phylogeny Programs </title>
<script data-dapp-detection="">!function(){let e=!1;function n(){if(!e){const n=document.createElement("meta");n.name="dapp-detected",document.head.appendChild(n),e=!0}}if(window.hasOwnProperty("ethereum")){if(window.__disableDappDetectionInsertion=!0,void 0===window.ethereum)return;n()}else{var t=window.ethereum;Object.defineProperty(window,"ethereum",{configurable:!0,enumerable:!1,set:function(e){window.__disableDappDetectionInsertion||n(),t=e},get:function(){if(!window.__disableDappDetectionInsertion){const e=arguments.callee;e&&e.caller&&e.caller.toString&&-1!==e.caller.toString().indexOf("getOwnPropertyNames")||n()}return t}})}}();</script></head>
<body bgcolor="#CCFFFF" data-new-gr-c-s-check-loaded="14.1022.0" data-gr-ext-installed="">
<div align="center">
<table width="80%" border="1"><tbody><tr><td bgcolor="#cce8ff">
<font size="+1">
As people can see from the dates on the most recent updates of these Phylogeny
Programs pages, I have not had time to keep them up-to-date since 2012.  I have
now retired, which gives me more time to do research and to support online
resources.  So I have hopes of resuming updates, fixing links, and catching up
with the field of phylogenetic inference.
<p>
I hope soon to move these software listings webpages to a Github archive, and invite
others to help contribute to them and maintain them.
</p></font><p><font size="+1">
In the meantime, I may not be able to devote time to searching for new 
programs, so <b>their authors are begged to (please!) use the submission form 
instead</b>.  That form will be found at the "Submitting" link below. If you 
are upset that your program is not included, but it's too much trouble for you 
to fill out the submission form, then I will not listen to you.  If anyone else wants to help with 
this, let me know.
</font>
</p></td></tr></tbody></table>
</div>
<p>
<table bgcolor="#ccffff" border="0" cellspacing="4" width="100%"><tbody><tr bgcolor="#cce0e0">
<td align="center" width="14%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.html#methods" style="text-decoration:none">Methods</a></td>
<td align="center" width="14%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.html#systems" style="text-decoration:none">By computer</a></td>
<td align="center" width="14%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.xref.html" style="text-decoration:none">Cross-referenced</a></td>
<td align="center" width="14%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.html#datatypes" style="text-decoration:none">Data types</a></td>
<td align="center" width="14%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.serv.html" style="text-decoration:none">Web servers</a></td>
<td align="center" width="14%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.html#recent" style="text-decoration:none">New programs</a></td>
<td align="center" width="14%" color="black"><a href="http://evolution.gs.washington.edu/cgi-bin/software-form-new1.pl" style="text-decoration:none">Submitting</a></td>
</tr></tbody></table>
</p><p>
</p><center>
<table>
<tbody><tr>
<td align="LEFT" valign="MIDDLE">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP"><img src="./Phylogeny Programs_files/PHYLIP.gif" alt="[Phylip icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP"><img src="./Phylogeny Programs_files/paupicon.gif" alt="[PAUP* icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade"><img src="./Phylogeny Programs_files/maccladeicon.gif" alt="[MacClade icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Hennig86"><img src="./Phylogeny Programs_files/hennig86.gif" alt="[Hennig86 icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Random%20Cladistics"><img src="./Phylogeny Programs_files/rcicon.gif" alt="[Random Cladistics icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#AutoDecay"><img src="./Phylogeny Programs_files/autodecay.gif" alt="[AutoDecay icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON"><img src="./Phylogeny Programs_files/treecon1.gif" alt="[TREECON icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectrum"><img src="./Phylogeny Programs_files/spectrum.gif" alt="[Spectrum icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree"><img src="./Phylogeny Programs_files/SplitsTree.gif" alt="[SplitsTree icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT"><img src="./Phylogeny Programs_files/cpwicon.gif" alt="[COMPONENT icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP"><img src="./Phylogeny Programs_files/treeicon.gif" alt="[TREEMAP icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC"><img src="./Phylogeny Programs_files/caic.gif" alt="[CAIC icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView"><img src="./Phylogeny Programs_files/treeview.gif" alt="[TreeView icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView"><img src="./Phylogeny Programs_files/treeviewx.gif" alt="[TreeView X icon here]"></a>
</td>
<td align="RIGHT" valign="MIDDLE">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DNA%20Stacks"><img src="./Phylogeny Programs_files/DNAStacks.gif" alt="[DNA Stacks icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DendroMaker"><img src="./Phylogeny Programs_files/dmicon.gif" alt="[DendroMaker icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE"><img src="./Phylogeny Programs_files/conserve.gif" alt="[CONSERVE icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc"><img src="./Phylogeny Programs_files/genedoc.gif" alt="[GeneDoc icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Seq-Gen"><img src="./Phylogeny Programs_files/seq-gen.gif" alt="[Seq-Gen icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup"><img src="./Phylogeny Programs_files/seqpup.gif" alt="[SeqPup icon]"></a>
</td>
<td align="LEFT" valign="MIDDLE">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDrawDeck"><img src="./Phylogeny Programs_files/treedrawdeck.gif" alt="[Tree Draw Deck icon]"></a>
</td>
</tr>
<tr>
<td align="RIGHT" valign="MIDDLE">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron"><img src="./Phylogeny Programs_files/phylodendron.gif" alt="[Phylodendron icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TreeRot"><img src="./Phylogeny Programs_files/treerot.gif" alt="[TreeRot icon]"></a>
</td>
<td align="LEFT" valign="MIDDLE">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treevolve"><img src="./Phylogeny Programs_files/treevolve.gif" alt="[Treevolve icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE"><img src="./Phylogeny Programs_files/puzzle.gif" alt="[TREE-PUZZLE icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PSeq-Gen"><img src="./Phylogeny Programs_files/PSeq-Gen.gif" alt="[PSeq-Gen icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GeneTree"><img src="./Phylogeny Programs_files/genetree.gif" alt="[GeneTree icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup"><img src="./Phylogeny Programs_files/seqpup-java.jpg" alt="[SeqPup Java icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#popgene"><img src="./Phylogeny Programs_files/popgene.gif" alt="[POPGENE icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate"><img src="./Phylogeny Programs_files/qdate.gif" alt="[QDate icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA"><img src="./Phylogeny Programs_files/tfpga.gif" alt="[TFPGA icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PASSML"><img src="./Phylogeny Programs_files/passml.gif" alt="[PASSML icon]"></a>
</td>
<td align="RIGHT" valign="MIDDLE">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#genetix"><img src="./Phylogeny Programs_files/genetix.gif" alt="[Genetix icon]"></a>
</td>
<td align="LEFT" valign="MIDDLE">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX"><img src="./Phylogeny Programs_files/TREX.gif" alt="[T-REX icon]"></a>
</td>
<td align="RIGHT">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NJplot"><img src="./Phylogeny Programs_files/NJplot.gif" alt="[NJPlot icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin"><img src="./Phylogeny Programs_files/arlequin.GIF" alt="[Arlequin icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE"><img src="./Phylogeny Programs_files/dambe.gif" alt="[DAMBE icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DnaSP"><img src="./Phylogeny Programs_files/dnasp.gif" alt="[DnaSP icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest"><img src="./Phylogeny Programs_files/Modeltest.gif" alt="[Modeltest icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CodonBootstrap"><img src="./Phylogeny Programs_files/CodonBoot.gif" alt="[CodonBootstrap icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS"><img src="./Phylogeny Programs_files/dnasis_b.jpg" alt="[DNASIS icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BioEdit"><img src="./Phylogeny Programs_files/bioedit.gif" alt="[BioEdit icon]"></a>
</td>
</tr>
<tr>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ProSeq"><img src="./Phylogeny Programs_files/proseq.gif" alt="[ProSeq icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MacroCAIC"><img src="./Phylogeny Programs_files/macrocaic.gif" alt="[MacroCAIC icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL"><img src="./Phylogeny Programs_files/pal.gif" alt="[PAL icon]"></a>
</td>
<td align="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon"><img src="./Phylogeny Programs_files/radcon.gif" alt="[RadCon icon]"></a>
</td>
<td align="RIGHT" valign="MIDDLE">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PI"><img src="./Phylogeny Programs_files/pi.gif" alt="[PI icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeEdit"><img src="./Phylogeny Programs_files/treedit.gif" alt="[TreeEdit icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate"><img src="./Phylogeny Programs_files/tipdate.gif" alt="[TipDate
icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO"><img src="./Phylogeny Programs_files/plato.gif" alt="[PLATO icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeExplorer"><img src="./Phylogeny Programs_files/TreeExplorer.gif" alt="[TreeExplorer icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY"><img src="./Phylogeny Programs_files/HyPhy.gif" alt="[HY-PHY icon]"></a>
</td>
<td colspan="1" align="CENTER">
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie"><img src="./Phylogeny Programs_files/genie.gif" alt="[Genie icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeThief"><img src="./Phylogeny Programs_files/treethief.gif" alt="[TreeThief icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS"><img src="./Phylogeny Programs_files/TCS.gif" alt="[TCS icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE"><img src="./Phylogeny Programs_files/edible.gif" alt="[EDIBLE icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot"><img src="./Phylogeny Programs_files/winboot.gif" alt="[Winboot icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA"><img src="./Phylogeny Programs_files/mega3.gif" alt="[Mega3 icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite"><img src="./Phylogeny Programs_files/mesquite.gif" alt="[Mesquite icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyledit"><img src="./Phylogeny Programs_files/phyledit.gif" alt="[Phyledit icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax"><img src="./Phylogeny Programs_files/syntax.gif" alt="[SYN-TAX icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP"><img src="./Phylogeny Programs_files/PTP.gif" alt="[PTP icon]"></a>
</td>
</tr>
<tr>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network"><img src="./Phylogeny Programs_files/network.gif" alt="[Network icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST"><img src="./Phylogeny Programs_files/past.gif" alt="[PAST icon]"></a>
</td>
<td colspan="17" rowspan="2" align="CENTER">
<img src="./Phylogeny Programs_files/programs2.gif" alt="PHYLOGENY PROGRAMS">
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeneStudioPro"><img src="./Phylogeny Programs_files/gspro.gif" alt="[GeneStudio Pro icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder"><img src="./Phylogeny Programs_files/treefinder.gif" alt="[Treefinder icon]"></a>
</td>
</tr>
<tr>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW"><img src="./Phylogeny Programs_files/clustalx.gif" alt="[ClustalX icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP"><img src="./Phylogeny Programs_files/treemap2.gif" alt="[Treemap2.0beta icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes"><img src="./Phylogeny Programs_files/mrbayes.gif" alt="[MrBayes icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA"><img src="./Phylogeny Programs_files/YCDMA.gif" alt="[YCDMA icon]"></a>
</td>
</tr>
<tr>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NSA"><img src="./Phylogeny Programs_files/NSA.gif" alt="[NSA icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST"><img src="./Phylogeny Programs_files/beast.gif" alt="[BEAST icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Tracer"><img src="./Phylogeny Programs_files/tracer.gif" alt="[Tracer icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylogenetic%20Investigator"><img src="./Phylogeny Programs_files/PI.jpg" alt="[Phylogenetic Investigator icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.serv.html#Datamonkey%20server"><img src="./Phylogeny Programs_files/datamonkey.gif" alt="[Datamonkey server icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE"><img src="./Phylogeny Programs_files/symmetree.gif" alt="[SymmeTREE icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer"><img src="./Phylogeny Programs_files/tj.gif" alt="[TreeJuxtaposer icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace"><img src="./Phylogeny Programs_files/jevtrace.gif" alt="[Jevtrace icon]"></a>
</td>
<td valign="TOP">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet"><img src="./Phylogeny Programs_files/spectronet.gif" alt="[Spectronet icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT"><img src="./Phylogeny Programs_files/tnt.jpg" alt="[TNT icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CAFCA"><img src="./Phylogeny Programs_files/cafca.gif" alt="[CAFCA icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen"><img src="./Phylogeny Programs_files/phylogen.gif" alt="[Phylogen icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree"><img src="./Phylogeny Programs_files/dnatree.gif" alt="[Dnatree icon]"></a>
</td>
<td>
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest"><img src="./Phylogeny Programs_files/prottest.jpg" alt="[ProtTest icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS"><img src="./Phylogeny Programs_files/geodis.jpg" alt="[GEODIS icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan"><img src="./Phylogeny Programs_files/treescan.jpg" alt="[TreeScan icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treeme"><img src="./Phylogeny Programs_files/Treeme.gif" alt="[TreeMe icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot"><img src="./Phylogeny Programs_files/simplot.jpg" alt="[Simplot icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist"><img src="./Phylogeny Programs_files/ProfDist.jpg" alt="[ProfDist icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START"><img src="./Phylogeny Programs_files/start.gif" alt="[START icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute"><img src="./Phylogeny Programs_files/permute.gif" alt="[Permute! icon]"></a>
</td>
</tr>
<tr>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuperTree"><img src="./Phylogeny Programs_files/SuperTree.gif" alt="[SuperTree icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Swaap"><img src="./Phylogeny Programs_files/Swaap.gif" alt="[Swaap icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Swaap"><img src="./Phylogeny Programs_files/SwaapPH.gif" alt="[Swaap PH icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA"><img src="./Phylogeny Programs_files/mesa2.jpg" alt="[MESA icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree"><img src="./Phylogeny Programs_files/NimbleTree.jpg" alt="[NimbleTree icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw"><img src="./Phylogeny Programs_files/arbodraw.gif" alt="[ArboDraw icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP"><img src="./Phylogeny Programs_files/simmap.jpg" alt="[SIMMAP icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SGRunner"><img src="./Phylogeny Programs_files/SGRunner.jpg" alt="[SGRunner icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer"><img src="./Phylogeny Programs_files/cbcanalyzer.gif" alt="[CBCAnalyzer icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung"><img src="./Phylogeny Programs_files/Notung.gif" alt="[Notung icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Cadence"><img src="./Phylogeny Programs_files/Cadence.jpg" alt="[Cadence icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker"><img src="./Phylogeny Programs_files/treemaker.gif" alt="[TreeMaker icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy"><img src="./Phylogeny Programs_files/baliphy.jpg" alt="[BAli-Phy icon]"></a>
</td>
<td valign="CENTER">
&nbsp;&nbsp;<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious"><img src="./Phylogeny Programs_files/geneious.gif" alt="[Geneious icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.serv.html#Phylemon_server"><img src="./Phylogeny Programs_files/phylemon.jpg" alt="[Phylemon server icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MrEnt"><img src="./Phylogeny Programs_files/mrent.gif" alt="[MrEnt icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan"><img src="./Phylogeny Programs_files/kakusan4.gif" alt="[Kakusan4 icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAPPS"><img src="./Phylogeny Programs_files/mapps.jpg" alt="[MAPPS icon]"></a>
</td>
<td valign="CENTER">
&nbsp;<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI"><img src="./Phylogeny Programs_files/garli.jpg" alt="[GARLI icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat"><img src="./Phylogeny Programs_files/treestat.gif" alt="[TreeStat icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree"><img src="./Phylogeny Programs_files/figtree.gif" alt="[FigTree icon]"></a>
</td>
</tr>
<tr>
<td valign="CENTER">
&nbsp;&nbsp;<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TopD/fMts"><img src="./Phylogeny Programs_files/topd.gif" alt="[TopD/fMts icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB"><img src="./Phylogeny Programs_files/lvb.gif" alt="[LVB icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree"><img src="./Phylogeny Programs_files/SuperTree(1).gif" alt="[Supertree icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#mavric"><img src="./Phylogeny Programs_files/mavric.gif" alt="[Mavric icon]"></a>
</td>
<td halign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree"><img src="./Phylogeny Programs_files/hypertree.gif" alt="[HyperTree icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope"><img src="./Phylogeny Programs_files/dendroscope.jpg" alt="[Dendroscope icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom"><img src="./Phylogeny Programs_files/phylocom.gif" alt="[Phylocom icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.serv.html#phylogeny.fr"><img src="./Phylogeny Programs_files/phylogeny-fr.gif" alt="[phylogeny.fr icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi"><img src="./Phylogeny Programs_files/topali.gif" alt="[TOPALi icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator"><img src="./Phylogeny Programs_files/ES.jpg" alt="[ES icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GenoDive"><img src="./Phylogeny Programs_files/genodive.gif" alt="[GenoDive icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.serv.html#swami"><img src="./Phylogeny Programs_files/swami.gif" alt="[swami icon]"></a>
</td>
<td valign="CENTER">
&nbsp;&nbsp;&nbsp;<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility"><img src="./Phylogeny Programs_files/phyutility.jpg" alt="[Phyutility icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget"><img src="./Phylogeny Programs_files/pw-icon.png" alt="[PhyloWidget icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics"><img src="./Phylogeny Programs_files/bn.gif" alt="[Bionumerics icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST"><img src="./Phylogeny Programs_files/best.jpg" alt="[BEST icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS"><img src="./Phylogeny Programs_files/mantis.gif" alt="[MANTiS icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.serv.html#synatree"><img src="./Phylogeny Programs_files/synatree1.gif" alt="[Synatree icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE"><img src="./Phylogeny Programs_files/phase.gif" alt="[PHASE icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE"><img src="./Phylogeny Programs_files/ugene.jpg" alt="[UGENE icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE"><img src="./Phylogeny Programs_files/ete.gif" alt="[ETE icon]"></a>
</td>
</tr>
<tr>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IDEA"><img src="./Phylogeny Programs_files/idea.jpg" alt="[IDEA icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DendroPy"><img src="./Phylogeny Programs_files/dendropy.gif" alt="[DendroPy icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#puzzleboot"><img src="./Phylogeny Programs_files/puzzleboot.gif" alt="[Puzzleboot icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloQuart"><img src="./Phylogeny Programs_files/phyloquart.jpg" alt="[PhyloQuart icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ELW"><img src="./Phylogeny Programs_files/elw.jpg" alt="[ELW icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie"><img src="./Phylogeny Programs_files/tinkertree.gif" alt="[Cactus-Pie icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA"><img src="./Phylogeny Programs_files/metapiga.jpg" alt="[Cactus-Pie icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque"><img src="./Phylogeny Programs_files/bosque.jpg" alt="[Cactus-Pie icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Likewind"><img src="./Phylogeny Programs_files/likewind.jpg" alt="[Cactus-Pie icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DAWG"><img src="./Phylogeny Programs_files/dawg.jpg" alt="[DAWG icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS"><img src="./Phylogeny Programs_files/gengis.gif" alt="[GenGIS icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph"><img src="./Phylogeny Programs_files/TreeGraph2.gif" alt="[GenGIS icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PC-ORD"><img src="./Phylogeny Programs_files/pc-ord.gif" alt="[GenGIS icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyRe"><img src="./Phylogeny Programs_files/phyre.jpg" alt="[PhyRe icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP"><img src="./Phylogeny Programs_files/iGTP.jpg" alt="[iGTP icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon"><img src="./Phylogeny Programs_files/recodon.jpg" alt="[Recodon icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon"><img src="./Phylogeny Programs_files/netrecodon.jpg" alt="[NetRecodon icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GHOSTS"><img src="./Phylogeny Programs_files/ghosts.jpg" alt="[GHOSTS icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.serv.html#NetTest"><img src="./Phylogeny Programs_files/nettest.jpg" alt="[NetTest icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#raxmlGUI"><img src="./Phylogeny Programs_files/raxmlgui.jpg" alt="[raxmlGUI icon]"></a>
</td>
<td valign="CENTER">
<a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree"><img src="./Phylogeny Programs_files/mixturetree.jpg" alt="[MixtureTree icon]"></a>
</td>
</tr>
</tbody></table>
</center>
<p>
<table bgcolor="#ccffff" border="0" cellspacing="4" width="100%"><tbody><tr bgcolor="#cce0e0">
<td align="center" width="16%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.html#changes" style="text-decoration:none">Changes</a></td>
<td align="center" width="16%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.html#new" style="text-decoration:none">Waiting list</a></td>
<td align="center" width="16%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.html#otherlists" style="text-decoration:none">Other lists</a></td>
<td align="center" width="16%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.old.html" style="text-decoration:none">Old programs</a></td>
<td align="center" width="16%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.not.html" style="text-decoration:none">Not listed</a></td>
<td align="center" width="16%" color="black"><a href="https://evolution.gs.washington.edu/phylip/software.html#news" style="text-decoration:none">News</a></td>
</tr></tbody></table>
</p><p>
Here are 392 phylogeny packages and 54 <a href="https://evolution.gs.washington.edu/phylip/software.serv.html#servers">free
web servers</a>, (almost) all that I know about.
It is an attempt to be completely comprehensive. I have not made any attempt to
exclude programs that do not meet some standard of quality or importance.
Updates to these pages are made roughly monthly. 
<a href="https://evolution.gs.washington.edu/phylip/software.html#new">Here</a> is a "waiting list" of new programs waiting to have
their full entries constructed. 
Many of the programs in these pages are available on the web, and some of
the older ones are also available from <a href="https://evolution.gs.washington.edu/phylip/software.serv.html#ftpservers">ftp server machines</a>.
</p><p>
The programs listed below include both free and non-free
ones; in some cases I do not know whether a program is free.  I have listed as
free those that I knew were free; for the others you have to ask their
distributor.  Usually when I say that a program is downloadable from a
web site, this means that it is available free.
</p><p>
Email addresses in these pages have had the <tt>@</tt> symbol replaced by
<tt> (at) </tt> and also surrounded by invisible confusing tags and blank
characters in hopes of foiling spambots that harvest email addresses.
</p><p>
If you discover any inaccuracies, or feel that I have left any important
programs or facts out, or if links do not work properly, please
e-mail me at:
(<code>joe<!--deathtospam-->&nbsp;(at)<!--deathtospam-->&nbsp;gs.washington.edu</code>).
You can also use the submission
form <a href="http://evolution.gs.washington.edu/cgi-bin/software-form-new1.pl">here</a> to submit new entries.

</p><p>
</p><div align="center">
<table border="1" width="80%">
<tbody><tr><td>
Owing to past NSF support of these pages, I am required to note that
any opinions, findings, and conclusions or recommendations expressed in this
material are those of the author and do not necessarily reflect the views of
the National Science Foundation (NSF supported these pages from 1995-2003).
</td></tr>
</tbody></table>
</div>
<p>

</p><center>
<table width="70%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
L</font><font size="+2" color="cyan">ist of packages arranged ...</font></strong></td>
</tr>
</tbody></table>
</center>
<p>
</p><h3>... <a href="https://evolution.gs.washington.edu/phylip/software.html#methods">by methods available</a></h3>
<h3>... <a href="https://evolution.gs.washington.edu/phylip/software.html#systems">by computer systems on which they work</a></h3>
<h3>... <a href="https://evolution.gs.washington.edu/phylip/software.xref.html">cross-referenced by method and by computer system.</a></h3>
<h3>... <a href="https://evolution.gs.washington.edu/phylip/software.html#datatypes">by ones which analyze particular kinds of data.</a></h3>
<h3>... <a href="https://evolution.gs.washington.edu/phylip/software.html#recent">to show the most recent listings </a></h3>
<h3>... <a href="https://evolution.gs.washington.edu/phylip/software.html#changes">to show ones most recently changed</a></h3>
<h3><a href="https://evolution.gs.washington.edu/phylip/software.old.html">Phylogeny programs formerly listed here</a> but no longer distributed</h3>
<h3><a href="https://evolution.gs.washington.edu/phylip/software.not.html">Which kinds of programs are and are not listed</a></h3>
<h2><a href="https://evolution.gs.washington.edu/phylip/software.html#otherlists">Other lists of phylogeny software</a></h2>
<h2><a href="https://evolution.gs.washington.edu/phylip/software.html#new">New programs waiting to be added</a></h2>
<p>

</p><hr>
<a name="methods">
<p>
</p><center>
<table width="80%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
T</font><font size="+2" color="cyan">able of contents by methods available</font></strong></td>
</tr>
</tbody></table>
</center>

<table><tbody><tr><td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#General"><strong>General-purpose packages</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Parsimony"><strong>Parsimony programs</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Distance"><strong>Distance matrix methods</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Distances"><strong>Computation of distances</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Likelihood"><strong>Maximum likelihood methods</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Bayesian"><strong>Bayesian inference methods</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Quartets"><strong>Quartets methods</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#AI"><strong>Artificial-intelligence and genetic algorithms methods</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Invariants"><strong>Invariants (or Evolutionary Parsimony) methods</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Interactive"><strong>Interactive tree manipulation</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Recombinant"><strong>Looking for hybridization or recombination events</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Bootstrapping"><strong>Bootstrapping and other measures of support</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Compatibility"><strong>Compatibility analysis</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Consensus"><strong>Consensus trees, subtrees, supertrees, distances between trees</strong></a>
</li></ul>
</td><td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Alignment"><strong>Tree-based alignment</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Genomics"><strong>Gene duplication and genomic analysis</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Biogeography"><strong>Biogeographic analysis and host-parasite comparison</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Comparative"><strong>Comparative method analysis</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Simulation"><strong>Simulation of trees or data</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Shapes"><strong>Examination of shapes of trees</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Stratigraphy"><strong>Clocks, dating and stratigraphy</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Modelselection"><strong>Model Selection</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Description"><strong>Description or prediction of data from trees</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Plotting"><strong>Tree plotting/drawing</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Management"><strong>Sequence management/job submission</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.html#Teaching"><strong>Teaching about phylogenies</strong></a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.serv.html"><strong>Web or e-mail servers that can analyze data for you</strong></a>
</li></ul>
</td></tr></tbody></table>
<p>
</p><hr>
</a><p><a name="methods">
</a><a name="General"><strong>General-purpose packages</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DendroPy">DendroPy</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
</li></ul>
<p>
<a name="Parsimony"><strong>Parsimony programs</strong></a>
</p><p>
<table border="0"><tbody><tr><td halign="LEFT">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Hennig86">Hennig86</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#RA">RA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#NONA">NONA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CAFCA">CAFCA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#sog">sog</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#gmaes">gmaes</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GeneTree">GeneTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MALIGN">MALIGN</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#POY">POY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GAPars">GAPars</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CRANN">CRANN</a>
</li></ul>
</td>
<td halign="LEFT" valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#FootPrinter">FootPrinter</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BPAnalysis">BPAnalysis</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Parsimov">Parsimov</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung">Notung</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IDEA">IDEA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PSODA">PSODA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#SeqState">SeqState</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloNet">PhyloNet</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Freqpars">Freqpars</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a>
</li></ul>
</td>
</tr></tbody></table>
</p><p>
<a name="Distance"><strong>Distance matrix methods</strong></a>
<table>
<tbody><tr>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MacT">MacT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ODEN">ODEN</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON">TREECON</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RESTSITE">RESTSITE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NTSYSpc">NTSYSpc</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#METREE">METREE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GDA">GDA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lintre">Lintre</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#gmaes">gmaes</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DENDRON">DENDRON</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#BIONJ">BIONJ</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA">TFPGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a>
</li></ul>
</td>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#sendbs">sendbs</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#nneighbor">nneighbor</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#weighbor">weighbor</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS">DNASIS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MINSPNET">MINSPNET</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#vCEBL">PEBBLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#qclust">qclust</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP">PTP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree">SplitsTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FastME">FastME</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MacVector">MacVector</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#QuickTree">QuickTree</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist">ProfDist</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START">START2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STC">STC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer">CBCAnalyzer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Rate4Site">Rate4Site</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IDEA">IDEA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PC-ORD">PC-ORD</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NINJA">NINJA</a>
</li></ul>
</td>
<td valign="top">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STATIO">Statio</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TIMER">TIMER</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Ancestor">Ancestor</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ANC-GENE">ANC-GENE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Bn-Bs">Bn-Bs</a>
</li></ul>
</td>
</tr>
</tbody></table>
</p><p>
<a name="Distances"><strong>Computation of distances</strong></a>
<table>
<tbody><tr>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RAPDistance">RAPDistance</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MULTICOMP">MULTICOMP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIPLOMO">DIPLOMO</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#OSA">OSA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RESTSITE">RESTSITE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NTSYSpc">NTSYSpc</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GCUA">GCUA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DERANGE2">DERANGE2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#popgene">POPGENE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA">TFPGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#REAP">REAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RSTCALC">RSTCALC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#genetix">Genetix</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTANCE">DISTANCE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#sendbs">sendbs</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DnaSP">DnaSP</a>
</li></ul>
</td>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#puzzleboot">puzzleboot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#qclust">qclust</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FSTAT">FSTAT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Phyltools">Phyltools</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA">YCDMA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NSA">NSA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#LDDist">LDDist</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DIVAGE">DIVAGE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Genepop">Genepop</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START">START2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Swaap">Swaap</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SwaapPH">Swaap PH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SPAGeDi">SPAGeDi</a>
</li></ul>
</td><td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer">CBCAnalyzer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rRNA">rRNA phylogeny</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GenoDive">GenoDive</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#analysis">analysis</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTREE">DISTREE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Bn-Bs">Bn-Bs</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#HON-new">HON-new</a>
</li></ul>
</td>
</tr> </tbody></table>
</p><p>
<a name="Likelihood"><strong>Maximum likelihood methods</strong></a>
</p><p>
<table><tbody><tr><td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAml">fastDNAml</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MOLPHY">MOLPHY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectrum">Spectrum</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree">SplitsTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PASSML">PASSML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">Modeltest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#dnarates">dnarates</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DT-ModSel">DT-ModSel</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAmlRev">fastDNAmlRev</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RevDNArates">RevDNArates</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rate-evolution">rate-evolution</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CONSEL">CONSEL</a>
</li></ul>
</td>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO">PLATO</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP">PTP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s-bootstrap">r8s-bootstrap</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BootPHYML">BootPHYML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARBOOT">PARBOOT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#p4">p4</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rhino">Rhino</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate">TipDate</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrAIC">MrAIC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modelfit">Modelfit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARAT">PARAT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIFRITZ">ALIFRITZ</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyNav">PhyNav</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DPRML">DPRML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MultiPhyl">MultiPhyl</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SSA">SSA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CoMET">CoMET</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mac5">Mac5</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYSIG">PHYSIG</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Rate4Site">Rate4Site</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#aLRT">aLRT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#McRate">McRate</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EREM">EREM</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PROCOV">PROCOV</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloCoCo">PhyloCoCo</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a>
</li></ul>
</td><td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#SeqState">SeqState</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Leaphy">Leaphy</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NHML">NHML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SLR">SLR</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rRNA">rRNA phylogeny</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Concaterpillar">Concaterpillar</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#NEPAL">NEPAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodeAxe">CodeAxe</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FastTree">FastTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#nhPhyML">nhPhyML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyML-Multi">PhyML-Multi</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#raxmlGUI">raxmlGUI</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree">MixtureTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GZ-Gamma">GZ-Gamma</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
</li></ul>
</td>
</tr>
</tbody></table>
</p><p>
<a name="Bayesian"><strong>Bayesian inference methods</strong></a>
</p><p>
<table><tbody><tr><td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAMBE">BAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesTreeScan">MrBayes tree scanners</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#p4">p4</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesPlugin">MrBayesPlugin</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloBayes">PhyloBayes</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Cadence">Cadence</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Multidivtime">Multidivtime</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST">BEST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#AMBIORE">AMBIORE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#bms_runner">bms_runner</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#burntrees">burntrees</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ANC-GENE">ANC-GENE</a>
</li></ul>
</td>
</tr>
</tbody></table>
</p><p>
<a name="Quartets"><strong>Quartets methods</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree">SplitsTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GEOMETRY">GEOMETRY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PICA">PICA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloQuart">PhyloQuart</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Willson">Willson quartets programs</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STC">STC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#LEVEL2">LEVEL2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
</li></ul>
<p>
<a name="AI"><strong>Artificial-intelligence and genetic algorithms methods</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP">PTP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GAPars">GAPars</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a>
</li></ul>
<p>
<a name="Invariants"><strong>Invariants (or Evolutionary Parsimony) methods</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li></ul>
<p>
<a name="Interactive"><strong>Interactive tree manipulation</strong></a>
<table border="0">
<tbody><tr>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PDAP">PDAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeTool">TreeTool</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#WINCLADA">WINCLADA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeEdit">TreeEdit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeExplorer">TreeExplorer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeThief">TreeThief</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon">RadCon</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#mavric">Mavric</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treeme">TreeMe</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw">ArboDraw</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung">Notung</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker">TreeMaker</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SimpleClade">SimpleClade</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Forest">Forest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
</li></ul>
</td></tr></tbody></table>
</p><p>
<a name="Recombinant"><strong>Looking for hybridization or recombination events</strong></a>
<table border="0">
<tbody><tr>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RecPars">RecPars</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#partimatrix">partimatrix</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO">PLATO</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START">START2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Likewind">Likewind</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DualBrothers">DualBrothers</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#cBrother">cBrother</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HGT">HGT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloNet">PhyloNet</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator">EvolSimulator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Concaterpillar">Concaterpillar</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyML-Multi">PhyML-Multi</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RDP3">RDP3</a>
</li></ul>
</td>
</tr>
</tbody></table>
</p><p>
<a name="Bootstrapping"><strong>Bootstrapping and other measures of
support</strong></a> (for Bayesian support methods see the <a href="https://evolution.gs.washington.edu/phylip/software.html#Bayesian">Bayesian
inference list</a> above).
<table>
<tbody><tr><td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Random%20Cladistics">Random Cladistics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#AutoDecay">AutoDecay</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TreeRot">TreeRot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DNA%20Stacks">DNA Stacks</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#OSA">OSA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lintre">Lintre</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#sog">sog</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARBOOT">PARBOOT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PICA">PICA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">ModelTest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TAXEQ3">TAXEQ3</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAMBE">BAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#puzzleboot">puzzleboot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CodonBootstrap">CodonBootstrap</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CONSEL">CONSEL</a>
</li></ul>
</td>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Phyltools">Phyltools</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s-bootstrap">r8s-bootstrap</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesTreeScan">MrBayes tree scanners</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BootPHYML">BootPHYML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute">Permute!</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ELW">ELW</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MultiPhyl">MultiPhyl</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GHOSTS">GHOSTS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#scaleboot">scaleboot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#aLRT">aLRT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloBayes">PhyloBayes</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloNet">PhyloNet</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FastTree">FastTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#burntrees">burntrees</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#raxmlGUI">raxmlGUI</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree">MixtureTree</a>
</li></ul>
</td></tr></tbody></table>
</p><p>
<a name="Compatibility"><strong>Compatibility analysis</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPROB">COMPROB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PICA">PICA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#partimatrix">partimatrix</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TIGER">TIGER</a>
</li></ul>
<p>
<a name="Consensus"><strong> Consensus trees, subtrees, supertrees, and distances between trees</strong></a>
</p><p>
<table border="0">
<tbody><tr>
<td valign="top">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP">TREEMAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NTSYSpc">NTSYSpc</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#REDCON">REDCON</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TAXEQ3">TAXEQ3</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon">RadCon</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RobinsonFoulds">Robinson and Foulds distance</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Clann">Clann</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyNav">PhyNav</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuperTree">SuperTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree_scripts">Supertree scripts</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree">Supertree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HeuristicMRF2">HeuristicMRF2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TopD/fMts">TopD/fMts</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rainbow">Rainbow</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhySIC">PhySIC_IST</a>
</li></ul>
</td>
<td valign="top">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloSort">PhyloSort</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#raxmlGUI">raxmlGUI</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
</li></ul>
</td>
</tr>
</tbody></table>
</p><p>
<a name="Alignment"><strong>Tree-based sequence alignment</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeAlign">TreeAlign</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW">ClustalW</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MALIGN">MALIGN</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc">GeneDoc</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#POY">POY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIGN">ALIGN</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS">DNASIS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#FootPrinter">FootPrinter</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIFRITZ">ALIFRITZ</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#T-Coffee">T-Coffee</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw">ArboDraw</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAFFT">MAFFT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software%20etc1.html#SATCHMO">LOBSTER</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MUSCLE">MUSCLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a>
</li></ul>
<p>
<a name="Genomics"><strong>Gene duplication and genomic analysis</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DERANGE2">DERANGE2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BPAnalysis">BPAnalysis</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung">Notung</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TopD/fMts">TopD/fMts</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DTscore">DTscore</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#gtp">gtp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator">EvolSimulator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DTdraw">DTdraw</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mgenome">Mgenome</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Tree%20Tracker">Tree Tracker</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#bms_runner">bms_runner</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS">MANTiS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS">MANTiS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a>
</li></ul>
<p>
<a name="Biogeography"><strong>Biogeographic analysis and host-parasite comparison</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP">TREEMAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DIVA">DIVA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeFitter">TreeFitter</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS">GEODIS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Tarzan">Tarzan</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ParaFit">ParaFit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#AxParafit">AxParafit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#S-DIVA">S-DIVA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lagrange">Lagrange</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CoRe-PA">CoRe-PA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Jane">Jane</a>
</li></ul>
<p>
<a name="Comparative"><strong>Comparative method analysis</strong></a>
<table>
<tbody><tr>
<td valign="TOP&quot;">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC">CAIC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PDAP">PDAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ANCML">ANCML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RIND">RIND</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MacroCAIC">MacroCAIC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PI">Phylogenetic Independence</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute">Permute!</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Parsimov">Parsimov</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIVERGE">DIVERGE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#IDC">IDC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#OUCH">OUCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYSIG">PHYSIG</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a>
</li></ul></td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#bms_runner">bms_runner</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SLOUCH">SLOUCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a>
</li></ul>
</td>
</tr>
</tbody></table>
</p><p>
<a name="Simulation"><strong>Simulation of trees or data</strong></a>
<table><tbody><tr>
<td valign="top">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Seq-Gen">Seq-Gen</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treevolve">Treevolve and PTreevolve</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PSeq-Gen">PSeq-Gen</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ROSE">ROSE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ProSeq">ProSeq</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen">Phylogen</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SGRunner">SGRunner</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#apTreeshape">apTreeshape</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Simprot">Simprot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EREM">EREM</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#indel-Seq-Gen">indel-Seq-Gen</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DAWG">DAWG</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolveAGene3">EvolveAGene3</a>
</li></ul>
</td>
<td valign="top">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator">EvolSimulator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon">Recodon</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon">NetRecodon</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a><br>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
</li></ul>
</td>
</tr></tbody></table>
</p><p>
<a name="Shapes"><strong>Examination of shapes of trees</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MacroCAIC">MacroCAIC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie">Genie</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon">RadCon</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#branchlength">BRANCHLENGTH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Tracer">Tracer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#apTreeshape">apTreeshape</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat">TreeStat</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#laser">laser</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyRe">PhyRe</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a>
</li></ul>
<p>
<a name="Stratigraphy"><strong>Clocks, dating and stratigraphy</strong></a>
<table><tbody><tr>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate">QDate</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">Modeltest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RRTree">RRTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#vCEBL">PEBBLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeEdit">TreeEdit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rate-evolution">rate-evolution</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#branchlength">BRANCHLENGTH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s">r8s</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate">TipDate</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rhino">Rhino</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GHOSTS">GHOSTS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Cadence">Cadence</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Multidivtime">Multidivtime</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodonRates">CodonRates</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PATHd8">PATHd8</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#McRate">McRate</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloBayes">PhyloBayes</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GRate">GRate</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NHML">NHML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TIMER">TIMER</a>
</li></ul>
</td>
</tr></tbody></table>
</p><p>
<a name="Modelselection"><strong>Model Selection</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">Modeltest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrAIC">MrAIC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modelfit">Modelfit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DT-ModSel">DT-ModSel</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAPPS">MAPPS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Concaterpillar">Concaterpillar</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STATIO">Statio</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#jMODELTEST">jMODELTEST</a>
</li></ul>
<p>
<a name="Description"><strong>Description or prediction of data from trees</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE">CONSERVE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDis">TreeDis</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RIND">RIND</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIVERGE">DIVERGE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a>
</li></ul>
<p>
<a name="Plotting"><strong>Tree plotting/drawing</strong></a>
<table><tbody><tr>
<td>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeTool">TreeTool</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NJplot">NJplot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DendroMaker">DendroMaker</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDrawDeck">Tree Draw Deck</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron">Phylodendron</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#unrooted">unrooted</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON">TREECON</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#mavric">Mavric</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeExplorer">TreeExplorer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeThief">TreeThief</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloTreeDraw">Phylogenetic Tree Drawing</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSetViz">TreeSetViz</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw">ArboDraw</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung">Notung</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DigTree">DigTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Paloverde">Paloverde</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MrEnt">MrEnt</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree">FigTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree">HyperTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeoPhyloBuilder">GeoPhyloBuilder</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSnatcher">TreeSnatcher Plus</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DTdraw">DTdraw</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#S-DIVA">S-DIVA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree">MixtureTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Archaeopteryx">Archaeopteryx</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a>
</li></ul>
</td>
</tr></tbody></table>
</p><p>
<a name="Management"><strong>Sequence management/job submission</strong></a>
</p><p>
<table>
<tbody><tr>
<td valign="top">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Random%20Cladistics">Random Cladistics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GDE">GDE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MUST">MUST 2000</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DNA%20Stacks">DNA Stacks</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARBOOT">PARBOOT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BioEdit">BioEdit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#WebPHYLIP">Singapore PHYLIP web interface</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#W2H">W2H</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyledit">Phyledit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeneStudioPro">GeneStudio Pro</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DPRML">DPRML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MBEToolbox">MBEToolbox</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PISE">PISE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a>
</li></ul>
</td>
<td valign="top">
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PyCogent">PyCogent</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYDIT">PHYDIT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a>
</li></ul>
</td></tr></tbody></table>
</p><p>
<a name="Teaching"><strong>Teaching about phylogenies</strong></a>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylogenetic%20Investigator">Phylogenetic Investigator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylap">Phylap</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree">Dnatree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SimpleClade">SimpleClade</a>
</li></ul>
<p>
</p><hr>
<p>
<a name="systems">
</a></p><p>
</p><center>
<table width="70%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
T</font><font size="+2" color="cyan">able of contents by computer systems<br>
on which they work</font></strong></td>
</tr>
</tbody></table>
</center>
<p>
</p><h3><a href="https://evolution.gs.washington.edu/phylip/software.html#Unix">Unix (source code in C or executables)</a></h3>
<h3><a href="https://evolution.gs.washington.edu/phylip/software.html#interpreters">Multiplatform interpreters (Java, Perl, Python, R,
MATLAB)</a></h3><h3>
</h3><h3><a href="https://evolution.gs.washington.edu/phylip/software.html#PCs">PC's</a></h3>
<h3> ... <a href="https://evolution.gs.washington.edu/phylip/software.html#Windows">under Windows</a></h3>
<h3> ... <a href="https://evolution.gs.washington.edu/phylip/software.html#DOS">under DOS or in a Windows Command tool "DOS box"</a></h3>
<h3><a href="https://evolution.gs.washington.edu/phylip/software.html#Macintosh">Macintoshes (under Mac OS X or Mac OS)</a></h3>
<h3><a href="https://evolution.gs.washington.edu/phylip/software.serv.html"><strong>e-mail or Web servers</strong> that can analyze data for you</a></h3>
<p>
</p><hr>
<p>
</p><ul>
<li><a name="Unix"><strong>Unix (source code in C or executables).</strong></a>
I have included programs that
are available as C source code because most Unix workstations have a C
compiler.  (Programs in other compiled languages such as FORTRAN and Pascal,
and in interpreted languages such as Java, Perl, Python, or R are also
included), as are Java executables.
For many of these the programs can also be compiled or run on Windows or
Mac OS X systems if they have the appropriate compilers or interpreters loaded.
<table>
<tbody><tr>
<td>
        <ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ODEN">ODEN</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lintre">Lintre</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#OSA">OSA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAml">fastDNAml</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MOLPHY">MOLPHY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree">SplitsTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeAlign">TreeAlign</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW">ClustalW</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MALIGN">MALIGN</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc">GeneDoc</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Seq-Gen">Seq-Gen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeTool">TreeTool</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GDE">GDE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#sog">sog</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron">Phylodendron</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treevolve">Treevolve and PTreevolve</a>
</li></ul>
</td>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PSeq-Gen">PSeq-Gen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#gmaes">gmaes</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GCUA">GCUA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DERANGE2">DERANGE2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#BIONJ">BIONJ</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ANCML">ANCML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate">QDate</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PASSML">PASSML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RecPars">RecPars</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARBOOT">PARBOOT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTANCE">DISTANCE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#sendbs">sendbs</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#partimatrix">partimatrix</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAMBE">BAMBE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#nneighbor">nneighbor</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#unrooted">unrooted</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ROSE">ROSE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#weighbor">weighbor</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloQuart">PhyloQuart</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#puzzleboot">puzzleboot</a>
</li></ul>
</td>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Willson">Willson quartets programs</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#POY">POY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RIND">RIND</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RRTree">RRTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#mavric">Mavric</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#dnarates">dnarates</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie">Genie</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#qclust">qclust</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAmlRev">fastDNAmlRev</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RevDNArates">RevDNArates</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#branchlength">BRANCHLENGTH</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CONSEL">CONSEL</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#W2H">W2H</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GAPars">GAPars</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s">r8s</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO">PLATO</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FastME">FastME</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloTreeDraw">Phylogenetic Tree Drawing</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s-bootstrap">r8s-bootstrap</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesTreeScan">MrBayes tree scanners</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RobinsonFoulds">Robinson and Foulds distance</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Clann">Clann</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BootPHYML">BootPHYML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#LDDist">LDDist</a>
</li></ul>
</td>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#p4">p4</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen">Phylogen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rhino">Rhino</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate">TipDate</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylap">Phylap</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree">Dnatree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#QuickTree">QuickTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#FootPrinter">FootPrinter</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BPAnalysis">BPAnalysis</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS">GEODIS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSetViz">TreeSetViz</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist">ProfDist</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrAIC">MrAIC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modelfit">Modelfit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARAT">PARAT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIFRITZ">ALIFRITZ</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyNav">PhyNav</a>
</li></ul>
</td>
</tr>
<tr><td>&nbsp;</td><td></td><td></td><td></td></tr>
<tr>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STC">STC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Likewind">Likewind</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ELW">ELW</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree_scripts">Supertree scripts</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Parsimov">Parsimov</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIVERGE">DIVERGE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#T-Coffee">T-Coffee</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer">CBCAnalyzer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GHOSTS">GHOSTS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Tarzan">Tarzan</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DT-ModSel">DT-ModSel</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DualBrothers">DualBrothers</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#apTreeshape">apTreeshape</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Multidivtime">Multidivtime</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mgenome">Mgenome</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ParaFit">ParaFit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#IDC">IDC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker">TreeMaker</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodonRates">CodonRates</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree">Supertree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#OUCH">OUCH</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DigTree">DigTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mac5">Mac5</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Paloverde">Paloverde</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HeuristicMRF2">HeuristicMRF2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CRANN">CRANN</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PATHd8">PATHd8</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAFFT">MAFFT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat">TreeStat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree">FigTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYSIG">PHYSIG</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#scaleboot">scaleboot</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#cBrother">cBrother</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesPlugin">MrBayesPlugin</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SATCHMO">LOBSTER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Rate4Site">Rate4Site</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TopD/fMts">TopD/fMts</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rainbow">Rainbow</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#McRate">McRate</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree">HyperTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloBayes">PhyloBayes</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Forest">Forest</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhySIC">PhySIC_IST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Simprot">Simprot</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST">BEST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EREM">EREM</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#indel-Seq-Gen">indel-Seq-Gen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MBEToolbox">MBEToolbox</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DTscore">DTscore</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PROCOV">PROCOV</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DAWG">DAWG</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#LEVEL2">LEVEL2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PSODA">PSODA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloSort">PhyloSort</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PISE">PISE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MUSCLE">MUSCLE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#AMBIORE">AMBIORE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HGT">HGT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NHML">NHML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SLR">SLR</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rRNA">rRNA phylogeny</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#AxParafit">AxParafit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolveAGene3">EvolveAGene3</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#gtp">gtp</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSnatcher">TreeSnatcher Plus</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator">EvolSimulator</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Concaterpillar">Concaterpillar</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DTdraw">DTdraw</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#NEPAL">NEPAL</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Tree%20Tracker">Tree Tracker</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#analysis">analysis</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodeAxe">CodeAxe</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FastTree">FastTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#laser">laser</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#bms_runner">bms_runner</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#burntrees">burntrees</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SLOUCH">SLOUCH</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS">MANTiS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Freqpars">Freqpars</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE">CONSERVE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
</li></ul>
</td>
</tr>
<tr><td>&nbsp;</td><td></td><td></td><td></td></tr>
<tr>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTREE">DISTREE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PyCogent">PyCogent</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DendroPy">DendroPy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC">CAIC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NINJA">NINJA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MUST">MUST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#nhPhyML">nhPhyML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyML-Multi">PhyML-Multi</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon">Recodon</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon">NetRecodon</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lagrange">Lagrange</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CoRe-PA">CoRe-PA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree">MixtureTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TIGER">TIGER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Jane">Jane</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GZ-Gamma">GZ-Gamma</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Archaeopteryx">Archaeopteryx</a>
<ul>
</ul></li></ul></td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Ancestor">Ancestor</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ANC-GENE">ANC-GENE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Bn-Bs">Bn-Bs</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#HON-new">HON-new</a>
</li></ul>
</td>
</tr></tbody></table>
<p>
</p></li><li><a name="interpreters"><strong>Multiplatform interpreters (Java, Perl,
Python, R, MATLAB)</strong></a>.  These are programs that are written in a
language that is interpreted, step by step, in real time by a computing
environment that can run on many different operating systems.
<p>
<i>(I am just starting to list interpreter code here.  Until recently it was
listed  under Unix, Windows and/or Mac OS X).  Until I finish transferring
interpreter code here this list will be incomplete, and you will find many
programs written in interpreted languages there). </i>
</p><p>
<strong>Java</strong>
</p><p>
</p><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#SeqState">SeqState</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IDEA">IDEA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloNet">PhyloNet</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung">Notung</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NINJA">NINJA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#qclust">qclust</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#vCEBL">PEBBLE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree">SplitsTree</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DPRML">DPRML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MultiPhyl">MultiPhyl</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloCoCo">PhyloCoCo</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CoMET">CoMET</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#jMODELTEST">jMODELTEST</a>
</li></ul>

<p>
</p></li><li><a name="PCs"><strong>PC's</strong></a>
  <ul>
     <li> <a name="Windows"><strong>as Windows executables</strong></a> (not
counting executing in a "DOS box").
Programs available as source code which is Windows-specific are listed here.
Java executables are also included.  (Note that compilers available on Windows
systems, particularly the free Cygwin and MinGW compilers, can also be used to
compile many of the programs listed above under Unix generic source code).
Programs run in interpreted environments such as Perl, Python, R or MATLAB
can also be run under Windows if the proper environment is installed.  These
programs are listed above under Unix.
<table>
<tbody><tr>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON">TREECON</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GDA">GDA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MOLPHY">MOLPHY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc">GeneDoc</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP">TREEMAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RAPDistance">RAPDistance</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron">Phylodendron</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#popgene">POPGENE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA">TFPGA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GeneTree">GeneTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RSTCALC">RSTCALC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#genetix">Genetix</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NJplot">NJplot</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#unrooted">unrooted</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DnaSP">DnaSP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a>
</li></ul>
</td>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS">DNASIS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MINSPNET">MINSPNET</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BioEdit">BioEdit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ProSeq">ProSeq</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#WINCLADA">WINCLADA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#NONA">NONA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PI">Phylogenetic Independence</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeExplorer">TreeExplorer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie">Genie</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s">r8s</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyledit">Phyledit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP">PTP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DIVA">DIVA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeFitter">TreeFitter</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeneStudioPro">GeneStudio Pro</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Phyltools">Phyltools</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mgenome">Mgenome</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA">YCDMA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NSA">NSA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Clann">Clann</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen">Phylogen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylap">Phylap</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree">Dnatree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS">GEODIS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSetViz">TreeSetViz</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treeme">TreeMe</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist">ProfDist</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START">START2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STC">STC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Swaap">Swaap</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SwaapPH">Swaap PH</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIVERGE">DIVERGE</a>
</li></ul>
</td>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw">ArboDraw</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SPAGeDi">SPAGeDi</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer">CBCAnalyzer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DualBrothers">DualBrothers</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SSA">SSA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Multidivtime">Multidivtime</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ParaFit">ParaFit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#IDC">IDC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker">TreeMaker</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodonRates">CodonRates</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DigTree">DigTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mac5">Mac5</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MrEnt">MrEnt</a>
</li></ul>
</td>
</tr>
<tr><td>&nbsp;</td><td></td><td></td><td></td></tr>
<tr>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SimpleClade">SimpleClade</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CRANN">CRANN</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PATHd8">PATHd8</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAFFT">MAFFT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat">TreeStat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree">FigTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesPlugin">MrBayesPlugin</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SATCHMO">LOBSTER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Rate4Site">Rate4Site</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rainbow">Rainbow</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#aLRT">aLRT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#McRate">McRate</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree">HyperTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeoPhyloBuilder">GeoPhyloBuilder</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Simprot">Simprot</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST">BEST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a>
</li></ul>
</td>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EREM">EREM</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MBEToolbox">MBEToolbox</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DTscore">DTscore</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#LEVEL2">LEVEL2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PSODA">PSODA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloSort">PhyloSort</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MUSCLE">MUSCLE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Leaphy">Leaphy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GRate">GRate</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SLR">SLR</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rRNA">rRNA phylogeny</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#AxParafit">AxParafit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DTscore">DTscore</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PROCOV">PROCOV</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE">CONSERVE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DAWG">DAWG</a>
</li></ul>
</td>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolveAGene3">EvolveAGene3</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSnatcher">TreeSnatcher Plus</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FastTree">FastTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS">MANTiS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PC-ORD">PC-ORD</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#S-DIVA">S-DIVA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYDIT">PHYDIT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RDP3">RDP3</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyRe">PhyRe</a>
</li></ul>
</td>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon">Recodon</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon">NetRecodon</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CoRe-PA">CoRe-PA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TIGER">TIGER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Jane">Jane</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TIMER">TIMER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#HON-new">HON-new</a>
</li></ul>
</td>
</tr>
</tbody></table>
<p>
     </p></li><li><a name="DOS"><strong>under DOS</strong></a> (MSDOS, PCDOS) or in a
Command Prompt accessory "DOS box" in Windows
<table>
<tbody><tr>
<td>
        <ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Hennig86">Hennig86</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#RA">RA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#NoNA">NONA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON">TREECON</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RESTSITE">RESTSITE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NTSYSpc">NTSYSpc</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#METREE">METREE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RAPDistance">RAPDistance</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIPLOMO">DIPLOMO</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW">ClustalW</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MALIGN">MALIGN</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc">GeneDoc</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Random%20Cladistics">Random Cladistics</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GEOMETRY">GEOMETRY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PDAP">PDAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PICA">PICA</a>
</li></ul>
</td>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#REDCON">REDCON</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TAXEQ3">TAXEQ3</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#BIONJ">BIONJ</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ANCML">ANCML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#REAP">REAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lintre">Lintre</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#sendbs">sendbs</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#weighbor">weighbor</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#POY">POY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDis">TreeDis</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CONSEL">CONSEL</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FSTAT">FSTAT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FastME">FastME</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate">QDate</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RobinsonFoulds">Robinson and Foulds distance</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DIVAGE">DIVAGE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BPAnalysis">BPAnalysis</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Genepop">Genepop</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTREE">DISTREE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GZ-Gamma">GZ-Gamma</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STATIO">Statio</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Ancestor">Ancestor</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ANC-GENE">ANC-GENE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Bn-Bs">Bn-Bs</a>
        </li></ul>
  
</td>
</tr>
</tbody></table>
</li></ul>
<p>
</p></li><li><a name="Macintosh"><strong>Macintosh Mac OS X or Mac OS
executables</strong></a>.
Java executables are also included.
(Note that most of the programs in the Unix/Linux list above can also
be compiled for Mac OS X using the GCC compiler that is distributed with
Mac OS X).  Programs run in the interpreted environments Perl and Python
can also be run in Mac OS X as these are included in the Mac OS X distribution.
We have not yet included these in the list below but should.
R or MATLAB can also be run under Mac OS X if the proper environment is
installed.  These programs are listed above under Unix.
<table>
<tbody><tr>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MacT">MacT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAml">fastDNAml</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectrum">Spectrum</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#AutoDecay">AutoDecay</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CAFCA">CAFCA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW">ClustalW</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP">TREEMAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC">CAIC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Seq-Gen">Seq-Gen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE">CONSERVE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NJplot">NJplot</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DendroMaker">DendroMaker</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MUST">MUST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DNA%20Stacks">DNA Stacks</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylogenetic%20Investigator">Phylogenetic Investigator</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDrawDeck">Tree Draw Deck</a>
</li></ul>
</td>
<td>
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron">Phylodendron</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TreeRot">TreeRot</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treevolve">Treevolve and PTreevolve</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PSeq-Gen">PSeq-Gen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#BIONJ">BIONJ</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GCUA">GCUA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GeneTree">GeneTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate">QDate</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#unrooted">unrooted</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT Lite</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#weighbor">weighbor</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">Modeltest</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Willson">Willson quartets programs</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIGN">ALIGN</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CodonBootstrap">CodonBootstrap</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS">DNASIS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO">PLATO</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MacroCAIC">MacroCAIC</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon">RadCon</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeEdit">TreeEdit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a>
</li></ul>
</td>
<td valign="TOP">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeThief">TreeThief</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie">Genie</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s">r8s</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GDA">GDA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DIVA">DIVA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeFitter">TreeFitter</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RobinsonFoulds">Robinson and Foulds distance</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Clann">Clann</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a>
</li></ul>
</td>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BootPHYML">BootPHYML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MacVector">MacVector</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen">Phylogen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rhino">Rhino</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate">TipDate</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylap">Phylap</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree">Dnatree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS">GEODIS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSetViz">TreeSetViz</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist">ProfDist</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute">Permute!</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SGRunner">SGRunner</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DualBrothers">DualBrothers</a>
</li></ul>
</td>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Cadence">Cadence</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ParaFit">ParaFit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker">TreeMaker</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree">Supertree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mac5">Mac5</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Paloverde">Paloverde</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CRANN">CRANN</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAPPS">MAPPS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PATHd8">PATHd8</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAFFT">MAFFT</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat">TreeStat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree">FigTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesPlugin">MrBayesPlugin</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rainbow">Rainbow</a>
</li></ul>
</td>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree">HyperTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhySIC">PhySIC_IST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST">BEST</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#indel-Seq-Gen">indel-Seq-Gen</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloCoCo">PhyloCoCo</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#LEVEL2">LEVEL2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PSODA">PSODA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloSort">PhyloSort</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MUSCLE">MUSCLE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SLR">SLR</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#AxParafit">AxParafit</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolveAGene3">EvolveAGene3</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSnatcher">TreeSnatcher Plus</a>
</li></ul>
</td>
</tr>
<tr><td>&nbsp;</td><td></td><td></td><td></td></tr>
<tr>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GenoDive">GenoDive</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS">MANTiS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PyCogent">PyCogent</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DendroPy">DendroPy</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#NEPAL">NEPAL</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon">Recodon</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lagrange">Lagrange</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon">NetRecodon</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CoRe-PA">CoRe-PA</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TIGER">TIGER</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a>
</li></ul>
</td>
<td valign="top">
<ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Jane">Jane</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a>
        </li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a>
</li></ul>
</td>
</tr>
</tbody></table>
</li></ul>
<p>
</p><hr>
<a name="datatypes">
<p>
</p><center>
<table width="70%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
A</font><font size="+2" color="cyan">nalyzing particular types of data</font></strong></td>
</tr>
</tbody></table>
</center>
<p>
Here you will find lists of programs that analyze types of data other than
molecular sequence data.  We will gradually expand this list of data types.
</p><p>
<table><tbody><tr>
<td valign="top">
<b>Microsatellite data</b>&nbsp;&nbsp;&nbsp;&nbsp;
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RSTCALC">RSTCALC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA">YCDMA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a>
</li></ul>
</td>
<td valign="top">
<b>RAPDs, RFLPs, or AFLPs</b>&nbsp;&nbsp;&nbsp;&nbsp;
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA">tfpga</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RAPDistance">RAPDistance</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#REAP">REAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RESTSITE">RESTSITE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DENDRON">DENDRON</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Phyltools">Phyltools</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li></ul>
</td>
<td valign="top">
<b>Continuous quantitative</b>&nbsp;&nbsp;&nbsp;&nbsp;<br>
<b>characters</b>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ANCML">ANCML</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PDAP">PDAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PI">Phylogenetic Independence</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC">CAIC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#IDC">IDC</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CoMET">CoMET</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#OUCH">OUCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYSIG">PHYSIG</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute">Permute</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SLOUCH">SLOUCH</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PC-ORD">PC-ORD</a>
</li></ul>
</td>
<td valign="top">
<b>Gene frequencies</b><br>
(aside from microsatellite loci)
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Freqpars">Freqpars</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GDA">GDA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#popgene">POPGENE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA">YCDMA</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FSTAT">FSTAT</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DnaSP">DnaSP</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DIVAGE">DIVAGE</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Genepop">Genepop</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SPAGeDi">SPAGeDi</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GenoDive">GenoDive</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a>
</li><li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a>
</li></ul>
</td>
</tr>
</tbody></table>
</p><p>
(under construction: more coming soon)
</p><p>
</p><hr>
</a><a name="recent">
<p>
</p><center>
<table width="70%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
R</font><font size="+2" color="cyan">ecent listings</font></strong></td>
</tr>
</tbody></table>
</center>
</a><p><a name="recent">
<strong>Here are the packages that have most recently been added to these listings:</strong> (the most recent ones first).  Entries are retained in this list
for 6 months.  Note also </a><a href="https://evolution.gs.washington.edu/phylip/software.html#new">below</a> the "waiting list" area
listing programs that are to be added.
You can use the submission form
<a href="http://evolution.gs.washington.edu/cgi-bin/software-form-new1.pl">here</a> to submit new entries. </p><p>
<table><tbody><tr>
<td>
<ul>
<li> (List is currently empty because I have been unable to do much updating
owing to other pressures so no new packages have been added in the last year).
</li></ul>
</td>
</tr></tbody></table>
</p><p>

</p><hr>
<a name="changes">
<p>
</p><center>
<table width="70%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
R</font><font size="+2" color="cyan">ecent changes</font></strong></td>
</tr>
</tbody></table>
</center>
<p>
<strong>Here are the packages whose entries have most recently been changed:</strong>
The date on which each change was entered is shown.  Entries are retained in
this list for 6 months.
(Note that changes may be as small as updated version numbers or a
modified web address). The most recent changes are first. 
</p><p>
<table border="0"><tbody><tr><td>
</td></tr><tr><td>
<ul>
<li> <a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a> (19 June 2014)
</li></ul>
</td></tr></tbody></table>
</p><p>

</p><hr>
</a><a name="otherlists">
<p>
</p><center>
<table width="80%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
O</font><font size="+2" color="cyan">ther lists of phylogeny software</font></strong></td>
</tr>
</tbody></table>
</center>
<p>
</p></a><ul><a name="otherlists">
</a><li><a name="otherlists">There is one phylogeny software list even more complete and up-to-date
than this one: a more recent version of this list.  If you are
reading this on the web pages at our server
<tt>evolution.gs.washington.edu</tt>, you are reading the most up-to-date
version.  But if you are reading a version stored anywhere else, you might
want to </a><a href="http://evolution.gs.washington.edu/phylip/software.html">
look here</a> instead.
</li><li> Sergios-Orestis Kolokotronis has posted
<a href="http://research.amnh.org/users/koloko/softlinks/phylogeny.html">an extensive table</a>
of phylogeny programs at his site at the American Museum of Natural History,
and near it are others under headings such as "molecular evolution" and
"alignment".
</li><li> Wikipedia has a good list of sequence alignment software (including
both tree-based and non-tree-based alignment methods)
<a href="http://en.wikipedia.org/wiki/List_of_sequence_alignment_software">here</a>
at <code>http://en.wikipedia.org/wiki/List_of_sequence_alignment_software</code>
</li><li>David Robertson of the Bioinformatics Education and Research at the
University of Manchester, England, maintains a very informative web site at
listing programs and their web sites that test for the presence of
recombination or hybridization events in DNA sequence data.  It lists some
programs that are covered here, and others that are outside the scope of
these web pages.  <a href="http://bioinf.man.ac.uk/robertson/recombination/programs.shtml">That site</a> is located at
<code>http://bioinf.man.ac.uk/robertson/recombination/programs.shtml</code>.
</li><li>  Mike Robeson at the University of Colorado maintains a page with
multiple programs listed as <a href="http://rintintin.colorado.edu/~robeson/Bioinformatics/OS%20X%20Software%20%26%20Links.html">Bioinformatics
software for Mac OS X</a>.
</li><li>The Bioinformatics Organization, a nonprofit group in Hudson, Massachusetts,
has posted the <a href="http://www.bioinformatics.org/"><b>bioinformatics.org</b></a>
web pages.  These offer a free membership and host open source software projects
in bioinformatics.  They also have a <a href="http://bioinformatics.org/software/mol_linux.php3">Molecular Linux</a> listing of Linux programs to carry out bioinformatics
tasks, which can be sorted by keywords.
</li><li> On Wikipedia there is a
<a href="http://en.wikipedia.org/wiki/List_of_phylogenetic_tree_visualization_software">List of phylogenetic tree visualization software</a> at
<code>http://en.wikipedia.org/wiki/List_of_phylogenetic_tree_visualization_software</code>
</li><li> The University of California Museum of Paleontology page of
<a href="http://www.ucmp.berkeley.edu/subway/phylo/phylosoft.html">
Phylogenetics Software Resources</a> at
<code>http://www.ucmp.berkeley.edu/subway/phylo/phylosoft.html</code>.
A few programs are listed, but there is
a very nice list of software lists there.
</li><li>Andrea Ramge, of Biomax Informatics AG, Martinsried, Germany
has created the <strong><code>bioinformatik.de</code></strong>
index of resources.  It includes a
<a href="http://www.bioinformatik.de/cgi-bin/browse/Catalog/Software/">
list of software</a> located at
<code>http://www.bioinformatik.de/cgi-bin/browse/Catalog/Software</code>.
The phylogeny programs listings there are located within the categories
for different operating systems.  The phylogeny software is under "Phylogenetic
Analysis" within each operating system.
</li><li> Richard Christen at the Université de Nice, France, has a list of
<a href="http://bioinfo.unice.fr/biodiv/Tree_editors.html">Tree and Tree-software for visualisation and manipulations dealing with phylogenetic trees</a> at
<code>http://bioinfo.unice.fr/biodiv/Tree_editors.html</code>
</li><li> Silvio Nihei at the University of São Paulo in Brazil has produced 
<a href="http://www.ib.usp.br/~silvionihei/programas.htm">a list: Programas para Filogenia</a> in Portugese.
It concentrates on a small number of programs that mostly use parsimony methods.
</li></ul>
<li>Don Gilbert, of the Department of Biology of the University of
Indiana, has a good web page on
<a href="http://iubio.bio.indiana.edu/soft/molbio/Listings.html">
Free Software in Molecular Biology for Macintosh and MS Windows computers </a>
as of 1998 at <tt>http://iubio.bio.indiana.edu/soft/molbio/Listings.html</tt>.
It lists some popular packages and all packages and programs kept
at the IUBio ftp server at that time (see <a href="https://evolution.gs.washington.edu/phylip/software.serv.html#ftpservers">
our description of that server</a>).  Unfortunately
the web links on that page are not active so the addresses must be
copied out and pasted into the URL box of your browser.
</li><li> Genamics, a company located in Hamilton, New Zealand, maintains the
<a href="http://genamics.com/software/index.htm">
SoftwareSeek searchable index of bioinformatics software</a>
at <code>http://genamics.com/software/index.htm</code>
in a number of categories.  One of them is Phylogenetic Analysis.  They have
a reasonably large number
of entries under that heading, though it also includes some statistical
genetics software that is really not phylogenetic.  Their listing has links
to the web sites of the software; for those programs that are not available
by Web they maintain copies for download at their server. Unfortunately,
the links to the software are
very out of date and few of them work.
<p>

</p><hr>
<a name="new">
<p>
</p><center>
<table width="80%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
N</font><font size="+2" color="cyan">ew programs waiting to be added</font></strong></td>
</tr>
</tbody></table>
</center>
</a><p><a name="new">
This is a "waiting list" showing links to the web pages of many new
phylogeny programs, which I have not yet had time to add to the main listing.
They will be listed there, with a single web link and
no detailed explanation.  I hope that this list will gradually shrink as the new
programs are put into the main listing.
You can use the submission form </a><a href="http://evolution.gs.washington.edu/cgi-bin/software-form.pl">here</a> to submit new entries. </p><p>
</p><p>
<b>These are waiting to be added:</b>
</p><p>
This bunch are all old programs from Masatoshi Nei's lab, available from the
same web page there
</p><ul>
<li> <a href="http://homes.bio.psu.edu/people/faculty/nei/software.htm">ADAPTSITE</a>,
intended to estimate positive and negative selection at a single amino
acid site.
</li></ul>
These are not from the Nei site:
<ul>
<li> <a href="http://www.mozoolab.net/index.php/software-download.html">PhyRe</a>
infers adequacy of taxon sampling for phylogenetic studies.
</li><li> <a href="http://cran.r-project.org/web/packages/phylobase/index.html">phylobase</a>
is an R package that contains a class of functions for comparative methods, incorporating one
or more trees and trait data.
</li><li> <a href="http://www.atgc-montpellier.fr/models/index.php?model=mixture">PhyML-mixtures</a>, a PhyML version for mixture of amino acid models (EX2, EX3, EHO, UL2, and UL3).
</li><li> <a href="http://www.atgc-montpellier.fr/phyd/">PhyD*</a>, Fast NJ-like
algorithms to deal with incomplete distance matrices.
</li><li> <a href="http://www.atgc-montpellier.fr/sdm/">SDM</a> a fast
distance-based approach for tree and supertree building in phylogenomics.
</li><li> <a href="http://www.atgc-montpellier.fr/ssimul/">SSIMUL</a> does
speciation signal extraction from multigene families.
</li><li> <a href="http://bioinformatics.hungry.com/clearcut/">Clearcut</a> carries
out Relaxed Neighbor Joining (RNJ), a faster NJ-like distance method.
</li><li> <a href="http://code.google.com/p/mp-est/">MP-EST</a> (also
described <a href="http://cars.desu.edu/faculty/lliu/research/mpest.html">here</a>) uses trees from
different loci to infer a species tree by a pseudo-maximum-likelihood method.
</li><li> <a href="https://stat.ethz.ch/pipermail/r-sig-phylo/2010-October/000816.html">TreeRogue</a>,
an R script for getting trees from published figures of them.
</li><li> <a href="http://biorg.cis.fiu.edu/SNE/">Serial NetEvolve</a> simulation
program evolves serially-sampled sequences with or without recombination.
</li><li> <a href="http://bibiserv.techfak.uni-bielefeld.de/rococo/">Rococo</a> reconstructs ancestral gene clusters for a multigene family
on a given tree.
</li></ul>
I will be adding new programs as they come in, and removing programs from this
waiting list as they are added to the main listing.
You can speed up the listing of a program
and help me by using the submission form
<a href="http://evolution.gs.washington.edu/cgi-bin/software-form-new1.pl">here</a> to submit entries. Any entries submitted that way (including ones that are
already in the Waiting List) will "jump the queue" and
get highest priority for being added to the full listing.<p>
</p><p>
</p><hr>
<a name="news">
<p>
</p><center>
<table width="80%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
N</font><font size="+2" color="cyan">ews</font></strong></td>
</tr>
</tbody></table>
</center>
<p>
</p><ul>
<li> (27 May 2012)  Adding one entry from the Waiting List each week, have
caught up with submissions submitted through the web form, and am populating
a new column in the cross-referenced table, one for multiplatform interpreted
code such as Java, Perl, Python, R, and MatLab.  The Java programs are
entered in that column.  After this I will put the others in it, then go back
to the main Phylogeny Programs front web page and put in a software category
for these interpreters.  If you have a new program, or an old one that I don't
list, don't wait for me to find it by myself -- I still don't have time to, so
use the web submission form.
</li><li> (19 March 2012)  Well, I should have known that another quarter of
teaching lay ahead.  Now that is done and I should make gradual progress over
the next two quarters.
</li><li> (23 December 2011)  Once again I got stalled by heavy teaching.  Now
resuming again and hope to gradually catch up.  One puzzle is how to handle
R packages.  There are a great many of them, and most are not listed here
yet.  It simply is too much work for me to track them all down, figure out their
features, and make an entry for each one.  So I will put in only those whose
authors use our submission form to help create the entry.
</li><li> (20 August 2011)  I have resumed adding new submissions that people
sent in using our software submission forms -- there are about 10
waiting, some having waited for over 6 months.  Apologies for that, I was
busy teaching and desperately trying to write up old results.  I hope to
gradually add all 10 over the next month or so, one at a time.
</li><li> (17 December 2010) I have started (on this 107th anniversary of the
Wright Brothers' famous first flight) this News section of the page.
The current status is that I have completed (over the past 2-3 years)
a complete pass through the listings, updating them.  However of course
some may have become outdated since then.  Ahead is adding new entries,
of which I expect there to be 30-40.  I am caught up on entries that
were submitted by the web submission form.
</li><li> (19 December 2010) I have finished adding the ones that were already
in our Waiting List.   Now to take some of the approximately 40 leads that
I have accumulated and put entries for the relevant ones into the Waiting List.
</li><li> (19 June 2014)  Things have been stalled for some years but now I am
gradually (and slowly) resuming adding and correcting entries.  Please keep
using the software submission form, and please be patient.  I was stalled owing
to needing to write grants, and owing to not getting them so I now have no
programming assistance. Progress to being current will be slow.  If you are
impatient, how about volunteering to help?  We could set up some web site
accessible to a team.
</li></ul>
<p>
</p><center>
<table width="80%">
<tbody><tr align="CENTER">
<td bgcolor="NAVY"><strong><font size="+3" color="cyan">
M</font><font size="+2" color="cyan">ysteries you can help us solve</font></strong></td>
</tr>
</tbody></table>
</center>
<p>
<b>These haven't been entered</b> because I am not sure what they do and whether
it is covered by this list. <em>Anyone with relevant information should let me know.</em>
</p></a><ul><a name="news">
</a><li><a name="news"> </a><a href="http://groups.yahoo.com/group/SB1981/files/3item/">3item</a>
extracts 3-item statements from "areagrams", whatever that means, but can only
be accessed by joining their Yahoo group.  So I don't really know what it does.
</li><li> <a href="http://www.uhnresearch.ca/labs/tillier/software.htm">Dependency v2.1</a>
uses Multiple Interdependency to detect functional interactions between amino acids in proteins.
Does not seem to actually use a phylogeny.
</li><li> <a href="http://www.uhnresearch.ca/labs/tillier/software.htm">Codep</a>
Maximizes co-evolutionary interdependencies to discover interacting proteins.  Also does not
seem to actually use a phylogeny.
</li><li><a href="http://www.atgc.org/PhyloGrapher/Graph_Ath_NBS.html">PhyloGrapher</a> shows clustering relationships between genes in a genome based on a
distance matrix.  But is it a phylogeny program?
</li><li><a href="http://www.genedata.com/products/phylosopher/index_eng.html">Phylosopher</a> commercial package for functional genomics said to include some phylogeny functions.  Does it?
</li><li><a href="http://www.multimedialibrary.com/Articles/Jack/Phylogenator.asp">Phylogenator</a> server displaying aligned sequences -- does it actually use or
construct phylogenies?
</li><li><a href="http://www.agapow.net/software/multilocus/">MultiLocus</a> calculates
multiple-locus measures of population differentiation from population genetic
data.  But unless someone can show me that it can calculate a measure of
distance between two populations within a data set, it does not seem appropriate
for this list.
</li><li><a href="http://www.phylo.org/sub_sections/software/">CIPRES-KEPLER</a>
Java-based framework for organizing workflow and submitting jobs.
I am not sure whether any specifically phylogeny-based pieces have yet been
supplied with this.
</li></ul>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Until I get  more information these programs cannot be further listed.
<p>

</p><hr>
<p>
<a href="https://evolution.gs.washington.edu/phylip/software.pars.html">To first page of main software listing</a>
</p><p>
</p><hr>
<a href="https://evolution.gs.washington.edu/phylip.html">
<img src="./Phylogeny Programs_files/PHYLIP.gif" alt="[Phylip icon here]"> ... to the PHYLIP home page</a>
<a name="bottom"><br></a>
<div> 
<p>
<font color="#666666">Notices added in compliance with University of Washington
requirements for web sites hosted at the University: &nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/privacy" target="_blank"><font color="#888888">Privacy</font></a>&nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.washington.edu/online/terms" target="_blank"><font color="#888888">Terms</font></a></font>
</p></div>


</li></body><grammarly-desktop-integration data-grammarly-shadow-root="true"></grammarly-desktop-integration></html>