<div data-align="center">

<table>
<colgroup>
<col style="width: 100%" />
</colgroup>
<tbody>
<tr class="odd">
<td>As people can see from the dates on the most recent updates of these Phylogeny Programs pages, I have not had time to keep them up-to-date since 2012. I have now retired, which gives me more time to do research and to support online resources. So I have hopes of resuming updates, fixing links, and catching up with the field of phylogenetic inference.
<p>I hope soon to move these software listings webpages to a Github archive, and invite others to help contribute to them and maintain them.</p>
<p>In the meantime, I may not be able to devote time to searching for new programs, so <strong>their authors are begged to (please!) use the submission form instead</strong>. That form will be found at the "Submitting" link below. If you are upset that your program is not included, but it's too much trouble for you to fill out the submission form, then I will not listen to you. If anyone else wants to help with this, let me know.</p></td>
</tr>
</tbody>
</table>

</div>

|                                                                             |                                                                                 |                                                                                   |                                                                                  |                                                                              |                                                                                 |                                                                                |
| :-------------------------------------------------------------------------: | :-----------------------------------------------------------------------------: | :-------------------------------------------------------------------------------: | :------------------------------------------------------------------------------: | :--------------------------------------------------------------------------: | :-----------------------------------------------------------------------------: | :----------------------------------------------------------------------------: |
| [Methods](https://evolution.gs.washington.edu/phylip/software.html#methods) | [By computer](https://evolution.gs.washington.edu/phylip/software.html#systems) | [Cross-referenced](https://evolution.gs.washington.edu/phylip/software.xref.html) | [Data types](https://evolution.gs.washington.edu/phylip/software.html#datatypes) | [Web servers](https://evolution.gs.washington.edu/phylip/software.serv.html) | [New programs](https://evolution.gs.washington.edu/phylip/software.html#recent) | [Submitting](http://evolution.gs.washington.edu/cgi-bin/software-form-new1.pl) |

[![\[Phylip
icon\]](./Phylogeny%20Programs_files/PHYLIP.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP)

[![\[PAUP\*
icon\]](./Phylogeny%20Programs_files/paupicon.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP)

[![\[MacClade
icon\]](./Phylogeny%20Programs_files/maccladeicon.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade)

[![\[Hennig86
icon\]](./Phylogeny%20Programs_files/hennig86.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#Hennig86)

[![\[Random Cladistics
icon\]](./Phylogeny%20Programs_files/rcicon.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#Random%20Cladistics)

[![\[AutoDecay
icon\]](./Phylogeny%20Programs_files/autodecay.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#AutoDecay)

[![\[TREECON
icon\]](./Phylogeny%20Programs_files/treecon1.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON)

[![\[Spectrum
icon\]](./Phylogeny%20Programs_files/spectrum.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectrum)

[![\[SplitsTree
icon\]](./Phylogeny%20Programs_files/SplitsTree.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree)

[![\[COMPONENT
icon\]](./Phylogeny%20Programs_files/cpwicon.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT)

[![\[TREEMAP
icon\]](./Phylogeny%20Programs_files/treeicon.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP)

[![\[CAIC
icon\]](./Phylogeny%20Programs_files/caic.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC)

[![\[TreeView
icon\]](./Phylogeny%20Programs_files/treeview.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView)

[![\[TreeView X icon
here\]](./Phylogeny%20Programs_files/treeviewx.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView)

[![\[DNA Stacks
icon\]](./Phylogeny%20Programs_files/DNAStacks.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#DNA%20Stacks)

[![\[DendroMaker
icon\]](./Phylogeny%20Programs_files/dmicon.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#DendroMaker)

[![\[CONSERVE
icon\]](./Phylogeny%20Programs_files/conserve.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE)

[![\[GeneDoc
icon\]](./Phylogeny%20Programs_files/genedoc.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc)

[![\[Seq-Gen
icon\]](./Phylogeny%20Programs_files/seq-gen.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Seq-Gen)

[![\[SeqPup
icon\]](./Phylogeny%20Programs_files/seqpup.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup)

[![\[Tree Draw Deck
icon\]](./Phylogeny%20Programs_files/treedrawdeck.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDrawDeck)

[![\[Phylodendron
icon\]](./Phylogeny%20Programs_files/phylodendron.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron)

[![\[TreeRot
icon\]](./Phylogeny%20Programs_files/treerot.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#TreeRot)

[![\[Treevolve
icon\]](./Phylogeny%20Programs_files/treevolve.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Treevolve)

[![\[TREE-PUZZLE
icon\]](./Phylogeny%20Programs_files/puzzle.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE)

[![\[PSeq-Gen
icon\]](./Phylogeny%20Programs_files/PSeq-Gen.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#PSeq-Gen)

[![\[GeneTree
icon\]](./Phylogeny%20Programs_files/genetree.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#GeneTree)

[![\[SeqPup Java
icon\]](./Phylogeny%20Programs_files/seqpup-java.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup)

[![\[POPGENE
icon\]](./Phylogeny%20Programs_files/popgene.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#popgene)

[![\[QDate
icon\]](./Phylogeny%20Programs_files/qdate.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate)

[![\[TFPGA
icon\]](./Phylogeny%20Programs_files/tfpga.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA)

[![\[PASSML
icon\]](./Phylogeny%20Programs_files/passml.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#PASSML)

[![\[Genetix
icon\]](./Phylogeny%20Programs_files/genetix.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#genetix)

[![\[T-REX
icon\]](./Phylogeny%20Programs_files/TREX.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX)

[![\[NJPlot
icon\]](./Phylogeny%20Programs_files/NJplot.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#NJplot)

[![\[Arlequin
icon\]](./Phylogeny%20Programs_files/arlequin.GIF)](https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin)

[![\[DAMBE
icon\]](./Phylogeny%20Programs_files/dambe.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE)

[![\[DnaSP
icon\]](./Phylogeny%20Programs_files/dnasp.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#DnaSP)

[![\[Modeltest
icon\]](./Phylogeny%20Programs_files/Modeltest.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest)

[![\[CodonBootstrap
icon\]](./Phylogeny%20Programs_files/CodonBoot.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#CodonBootstrap)

[![\[DNASIS
icon\]](./Phylogeny%20Programs_files/dnasis_b.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS)

[![\[BioEdit
icon\]](./Phylogeny%20Programs_files/bioedit.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#BioEdit)

[![\[ProSeq
icon\]](./Phylogeny%20Programs_files/proseq.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#ProSeq)

[![\[MacroCAIC
icon\]](./Phylogeny%20Programs_files/macrocaic.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#MacroCAIC)

[![\[PAL
icon\]](./Phylogeny%20Programs_files/pal.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#PAL)

[![\[RadCon
icon\]](./Phylogeny%20Programs_files/radcon.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon)

[![\[PI
icon\]](./Phylogeny%20Programs_files/pi.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#PI)

[![\[TreeEdit
icon\]](./Phylogeny%20Programs_files/treedit.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeEdit)

[![\[TipDate
icon\]](./Phylogeny%20Programs_files/tipdate.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate)

[![\[PLATO
icon\]](./Phylogeny%20Programs_files/plato.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO)

[![\[TreeExplorer
icon\]](./Phylogeny%20Programs_files/TreeExplorer.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeExplorer)

[![\[HY-PHY
icon\]](./Phylogeny%20Programs_files/HyPhy.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY)

[![\[Genie
icon\]](./Phylogeny%20Programs_files/genie.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie)

[![\[TreeThief
icon\]](./Phylogeny%20Programs_files/treethief.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeThief)

[![\[TCS
icon\]](./Phylogeny%20Programs_files/TCS.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#TCS)

[![\[EDIBLE
icon\]](./Phylogeny%20Programs_files/edible.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE)

[![\[Winboot
icon\]](./Phylogeny%20Programs_files/winboot.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot)

[![\[Mega3
icon\]](./Phylogeny%20Programs_files/mega3.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA)

[![\[Mesquite
icon\]](./Phylogeny%20Programs_files/mesquite.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite)

[![\[Phyledit
icon\]](./Phylogeny%20Programs_files/phyledit.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyledit)

[![\[SYN-TAX
icon\]](./Phylogeny%20Programs_files/syntax.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#syntax)

[![\[PTP
icon\]](./Phylogeny%20Programs_files/PTP.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP)

[![\[Network
icon\]](./Phylogeny%20Programs_files/network.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#Network)

[![\[PAST
icon\]](./Phylogeny%20Programs_files/past.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST)

![PHYLOGENY PROGRAMS](./Phylogeny%20Programs_files/programs2.gif)

[![\[GeneStudio Pro
icon\]](./Phylogeny%20Programs_files/gspro.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#GeneStudioPro)

[![\[Treefinder
icon\]](./Phylogeny%20Programs_files/treefinder.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder)

[![\[ClustalX
icon\]](./Phylogeny%20Programs_files/clustalx.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW)

[![\[Treemap2.0beta
icon\]](./Phylogeny%20Programs_files/treemap2.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP)

[![\[MrBayes
icon\]](./Phylogeny%20Programs_files/mrbayes.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes)

[![\[YCDMA
icon\]](./Phylogeny%20Programs_files/YCDMA.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA)

[![\[NSA
icon\]](./Phylogeny%20Programs_files/NSA.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#NSA)

[![\[BEAST
icon\]](./Phylogeny%20Programs_files/beast.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST)

[![\[Tracer
icon\]](./Phylogeny%20Programs_files/tracer.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#Tracer)

[![\[Phylogenetic Investigator
icon\]](./Phylogeny%20Programs_files/PI.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylogenetic%20Investigator)

[![\[Datamonkey server
icon\]](./Phylogeny%20Programs_files/datamonkey.gif)](https://evolution.gs.washington.edu/phylip/software.serv.html#Datamonkey%20server)

[![\[SymmeTREE
icon\]](./Phylogeny%20Programs_files/symmetree.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE)

[![\[TreeJuxtaposer
icon\]](./Phylogeny%20Programs_files/tj.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer)

[![\[Jevtrace
icon\]](./Phylogeny%20Programs_files/jevtrace.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace)

[![\[Spectronet
icon\]](./Phylogeny%20Programs_files/spectronet.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet)

[![\[TNT
icon\]](./Phylogeny%20Programs_files/tnt.jpg)](https://evolution.gs.washington.edu/phylip/software.pars.html#TNT)

[![\[CAFCA
icon\]](./Phylogeny%20Programs_files/cafca.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#CAFCA)

[![\[Phylogen
icon\]](./Phylogeny%20Programs_files/phylogen.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen)

[![\[Dnatree
icon\]](./Phylogeny%20Programs_files/dnatree.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree)

[![\[ProtTest
icon\]](./Phylogeny%20Programs_files/prottest.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest)

[![\[GEODIS
icon\]](./Phylogeny%20Programs_files/geodis.jpg)](https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS)

[![\[TreeScan
icon\]](./Phylogeny%20Programs_files/treescan.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan)

[![\[TreeMe
icon\]](./Phylogeny%20Programs_files/Treeme.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Treeme)

[![\[Simplot
icon\]](./Phylogeny%20Programs_files/simplot.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot)

[![\[ProfDist
icon\]](./Phylogeny%20Programs_files/ProfDist.jpg)](https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist)

[![\[START
icon\]](./Phylogeny%20Programs_files/start.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#START)

[![\[Permute\!
icon\]](./Phylogeny%20Programs_files/permute.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#Permute)

[![\[SuperTree
icon\]](./Phylogeny%20Programs_files/SuperTree.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#SuperTree)

[![\[Swaap
icon\]](./Phylogeny%20Programs_files/Swaap.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#Swaap)

[![\[Swaap PH
icon\]](./Phylogeny%20Programs_files/SwaapPH.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#Swaap)

[![\[MESA
icon\]](./Phylogeny%20Programs_files/mesa2.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA)

[![\[NimbleTree
icon\]](./Phylogeny%20Programs_files/NimbleTree.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree)

[![\[ArboDraw
icon\]](./Phylogeny%20Programs_files/arbodraw.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw)

[![\[SIMMAP
icon\]](./Phylogeny%20Programs_files/simmap.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP)

[![\[SGRunner
icon\]](./Phylogeny%20Programs_files/SGRunner.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#SGRunner)

[![\[CBCAnalyzer
icon\]](./Phylogeny%20Programs_files/cbcanalyzer.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer)

[![\[Notung
icon\]](./Phylogeny%20Programs_files/Notung.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung)

[![\[Cadence
icon\]](./Phylogeny%20Programs_files/Cadence.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#Cadence)

[![\[TreeMaker
icon\]](./Phylogeny%20Programs_files/treemaker.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker)

[![\[BAli-Phy
icon\]](./Phylogeny%20Programs_files/baliphy.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy)

  [![\[Geneious
icon\]](./Phylogeny%20Programs_files/geneious.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious)

[![\[Phylemon server
icon\]](./Phylogeny%20Programs_files/phylemon.jpg)](https://evolution.gs.washington.edu/phylip/software.serv.html#Phylemon_server)

[![\[MrEnt
icon\]](./Phylogeny%20Programs_files/mrent.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#MrEnt)

[![\[Kakusan4
icon\]](./Phylogeny%20Programs_files/kakusan4.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan)

[![\[MAPPS
icon\]](./Phylogeny%20Programs_files/mapps.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#MAPPS)

 [![\[GARLI
icon\]](./Phylogeny%20Programs_files/garli.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI)

[![\[TreeStat
icon\]](./Phylogeny%20Programs_files/treestat.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat)

[![\[FigTree
icon\]](./Phylogeny%20Programs_files/figtree.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree)

  [![\[TopD/fMts
icon\]](./Phylogeny%20Programs_files/topd.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TopD/fMts)

[![\[LVB
icon\]](./Phylogeny%20Programs_files/lvb.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#LVB)

[![\[Supertree
icon\]](./Phylogeny%20Programs_files/SuperTree\(1\).gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree)

[![\[Mavric
icon\]](./Phylogeny%20Programs_files/mavric.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#mavric)

[![\[HyperTree
icon\]](./Phylogeny%20Programs_files/hypertree.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree)

[![\[Dendroscope
icon\]](./Phylogeny%20Programs_files/dendroscope.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope)

[![\[Phylocom
icon\]](./Phylogeny%20Programs_files/phylocom.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom)

[![\[phylogeny.fr
icon\]](./Phylogeny%20Programs_files/phylogeny-fr.gif)](https://evolution.gs.washington.edu/phylip/software.serv.html#phylogeny.fr)

[![\[TOPALi
icon\]](./Phylogeny%20Programs_files/topali.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi)

[![\[ES
icon\]](./Phylogeny%20Programs_files/ES.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator)

[![\[GenoDive
icon\]](./Phylogeny%20Programs_files/genodive.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#GenoDive)

[![\[swami
icon\]](./Phylogeny%20Programs_files/swami.gif)](https://evolution.gs.washington.edu/phylip/software.serv.html#swami)

   [![\[Phyutility
icon\]](./Phylogeny%20Programs_files/phyutility.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility)

[![\[PhyloWidget
icon\]](./Phylogeny%20Programs_files/pw-icon.png)](https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget)

[![\[Bionumerics
icon\]](./Phylogeny%20Programs_files/bn.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics)

[![\[BEST
icon\]](./Phylogeny%20Programs_files/best.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST)

[![\[MANTiS
icon\]](./Phylogeny%20Programs_files/mantis.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS)

[![\[Synatree
icon\]](./Phylogeny%20Programs_files/synatree1.gif)](https://evolution.gs.washington.edu/phylip/software.serv.html#synatree)

[![\[PHASE
icon\]](./Phylogeny%20Programs_files/phase.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE)

[![\[UGENE
icon\]](./Phylogeny%20Programs_files/ugene.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE)

[![\[ETE
icon\]](./Phylogeny%20Programs_files/ete.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#ETE)

[![\[IDEA
icon\]](./Phylogeny%20Programs_files/idea.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#IDEA)

[![\[DendroPy
icon\]](./Phylogeny%20Programs_files/dendropy.gif)](https://evolution.gs.washington.edu/phylip/software.pars.html#DendroPy)

[![\[Puzzleboot
icon\]](./Phylogeny%20Programs_files/puzzleboot.gif)](https://evolution.gs.washington.edu/phylip/software.etc1.html#puzzleboot)

[![\[PhyloQuart
icon\]](./Phylogeny%20Programs_files/phyloquart.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloQuart)

[![\[ELW
icon\]](./Phylogeny%20Programs_files/elw.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#ELW)

[![\[Cactus-Pie
icon\]](./Phylogeny%20Programs_files/tinkertree.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie)

[![\[Cactus-Pie
icon\]](./Phylogeny%20Programs_files/metapiga.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA)

[![\[Cactus-Pie
icon\]](./Phylogeny%20Programs_files/bosque.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque)

[![\[Cactus-Pie
icon\]](./Phylogeny%20Programs_files/likewind.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Likewind)

[![\[DAWG
icon\]](./Phylogeny%20Programs_files/dawg.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#DAWG)

[![\[GenGIS
icon\]](./Phylogeny%20Programs_files/gengis.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS)

[![\[GenGIS
icon\]](./Phylogeny%20Programs_files/TreeGraph2.gif)](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph)

[![\[GenGIS
icon\]](./Phylogeny%20Programs_files/pc-ord.gif)](https://evolution.gs.washington.edu/phylip/software.dist.html#PC-ORD)

[![\[PhyRe
icon\]](./Phylogeny%20Programs_files/phyre.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyRe)

[![\[iGTP
icon\]](./Phylogeny%20Programs_files/iGTP.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP)

[![\[Recodon
icon\]](./Phylogeny%20Programs_files/recodon.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon)

[![\[NetRecodon
icon\]](./Phylogeny%20Programs_files/netrecodon.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon)

[![\[GHOSTS
icon\]](./Phylogeny%20Programs_files/ghosts.jpg)](https://evolution.gs.washington.edu/phylip/software.etc2.html#GHOSTS)

[![\[NetTest
icon\]](./Phylogeny%20Programs_files/nettest.jpg)](https://evolution.gs.washington.edu/phylip/software.serv.html#NetTest)

[![\[raxmlGUI
icon\]](./Phylogeny%20Programs_files/raxmlgui.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#raxmlGUI)

[![\[MixtureTree
icon\]](./Phylogeny%20Programs_files/mixturetree.jpg)](https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree)

|                                                                             |                                                                              |                                                                                    |                                                                              |                                                                            |                                                                       |
| :-------------------------------------------------------------------------: | :--------------------------------------------------------------------------: | :--------------------------------------------------------------------------------: | :--------------------------------------------------------------------------: | :------------------------------------------------------------------------: | :-------------------------------------------------------------------: |
| [Changes](https://evolution.gs.washington.edu/phylip/software.html#changes) | [Waiting list](https://evolution.gs.washington.edu/phylip/software.html#new) | [Other lists](https://evolution.gs.washington.edu/phylip/software.html#otherlists) | [Old programs](https://evolution.gs.washington.edu/phylip/software.old.html) | [Not listed](https://evolution.gs.washington.edu/phylip/software.not.html) | [News](https://evolution.gs.washington.edu/phylip/software.html#news) |

Here are 392 phylogeny packages and 54 [free web
servers](https://evolution.gs.washington.edu/phylip/software.serv.html#servers),
(almost) all that I know about. It is an attempt to be completely
comprehensive. I have not made any attempt to exclude programs that do
not meet some standard of quality or importance. Updates to these pages
are made roughly monthly.
[Here](https://evolution.gs.washington.edu/phylip/software.html#new) is
a "waiting list" of new programs waiting to have their full entries
constructed. Many of the programs in these pages are available on the
web, and some of the older ones are also available from [ftp server
machines](https://evolution.gs.washington.edu/phylip/software.serv.html#ftpservers).

The programs listed below include both free and non-free ones; in some
cases I do not know whether a program is free. I have listed as free
those that I knew were free; for the others you have to ask their
distributor. Usually when I say that a program is downloadable from a
web site, this means that it is available free.

Email addresses in these pages have had the `@` symbol replaced by ` 
(at)  ` and also surrounded by invisible confusing tags and blank
characters in hopes of foiling spambots that harvest email addresses.

If you discover any inaccuracies, or feel that I have left any important
programs or facts out, or if links do not work properly, please e-mail
me at: (`joe (at) gs.washington.edu`). You can also use the submission
form
[here](http://evolution.gs.washington.edu/cgi-bin/software-form-new1.pl)
to submit new entries.

<div data-align="center">

|                                                                                                                                                                                                                                                                                                            |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Owing to past NSF support of these pages, I am required to note that any opinions, findings, and conclusions or recommendations expressed in this material are those of the author and do not necessarily reflect the views of the National Science Foundation (NSF supported these pages from 1995-2003). |

</div>

|                                   |
| --------------------------------- |
| **List of packages arranged ...** |

### ... [by methods available](https://evolution.gs.washington.edu/phylip/software.html#methods)

### ... [by computer systems on which they work](https://evolution.gs.washington.edu/phylip/software.html#systems)

### ... [cross-referenced by method and by computer system.](https://evolution.gs.washington.edu/phylip/software.xref.html)

### ... [by ones which analyze particular kinds of data.](https://evolution.gs.washington.edu/phylip/software.html#datatypes)

### ... [to show the most recent listings](https://evolution.gs.washington.edu/phylip/software.html#recent)

### ... [to show ones most recently changed](https://evolution.gs.washington.edu/phylip/software.html#changes)

### [Phylogeny programs formerly listed here](https://evolution.gs.washington.edu/phylip/software.old.html) but no longer distributed

### [Which kinds of programs are and are not listed](https://evolution.gs.washington.edu/phylip/software.not.html)

## [Other lists of phylogeny software](https://evolution.gs.washington.edu/phylip/software.html#otherlists)

## [New programs waiting to be added](https://evolution.gs.washington.edu/phylip/software.html#new)

-----

<span id="methods"></span>

|                                            |
| ------------------------------------------ |
| **Table of contents by methods available** |

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#General"><strong>General-purpose packages</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Parsimony"><strong>Parsimony programs</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Distance"><strong>Distance matrix methods</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Distances"><strong>Computation of distances</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Likelihood"><strong>Maximum likelihood methods</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Bayesian"><strong>Bayesian inference methods</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Quartets"><strong>Quartets methods</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#AI"><strong>Artificial-intelligence and genetic algorithms methods</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Invariants"><strong>Invariants (or Evolutionary Parsimony) methods</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Interactive"><strong>Interactive tree manipulation</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Recombinant"><strong>Looking for hybridization or recombination events</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Bootstrapping"><strong>Bootstrapping and other measures of support</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Compatibility"><strong>Compatibility analysis</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Consensus"><strong>Consensus trees, subtrees, supertrees, distances between trees</strong></a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Alignment"><strong>Tree-based alignment</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Genomics"><strong>Gene duplication and genomic analysis</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Biogeography"><strong>Biogeographic analysis and host-parasite comparison</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Comparative"><strong>Comparative method analysis</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Simulation"><strong>Simulation of trees or data</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Shapes"><strong>Examination of shapes of trees</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Stratigraphy"><strong>Clocks, dating and stratigraphy</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Modelselection"><strong>Model Selection</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Description"><strong>Description or prediction of data from trees</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Plotting"><strong>Tree plotting/drawing</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Management"><strong>Sequence management/job submission</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.html#Teaching"><strong>Teaching about phylogenies</strong></a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.serv.html"><strong>Web or e-mail servers that can analyze data for you</strong></a></li>
</ul></td>
</tr>
</tbody>
</table>

-----

<span id="methods"></span> <span id="General">**General-purpose
packages**</span>

  - [PHYLIP](https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP)
  - [PAUP\*](https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP)
  - [MEGA](https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA)
  - [Phylo\_win](https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win)
  - [ARB](https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB)
  - [DAMBE](https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE)
  - [PAL](https://evolution.gs.washington.edu/phylip/software.pars.html#PAL)
  - [Bionumerics](https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics)
  - [Mesquite](https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite)
  - [PaupUp](https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp)
  - [BIRCH](https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH)
  - [Bosque](https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque)
  - [EMBOSS](https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS)
  - [phangorn](https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn)
  - [Bio++](https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++)
  - [ETE](https://evolution.gs.washington.edu/phylip/software.pars.html#ETE)
  - [DendroPy](https://evolution.gs.washington.edu/phylip/software.pars.html#DendroPy)
  - [SeaView](https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView)
  - [Crux](https://evolution.gs.washington.edu/phylip/software.pars.html#Crux)

<span id="Parsimony">**Parsimony programs**</span>

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Hennig86">Hennig86</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#RA">RA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#NONA">NONA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CAFCA">CAFCA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#sog">sog</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#gmaes">gmaes</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GeneTree">GeneTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MALIGN">MALIGN</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#POY">POY</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GAPars">GAPars</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CRANN">CRANN</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#FootPrinter">FootPrinter</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BPAnalysis">BPAnalysis</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Parsimov">Parsimov</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung">Notung</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IDEA">IDEA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PSODA">PSODA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#SeqState">SeqState</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloNet">PhyloNet</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Freqpars">Freqpars</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Distance">**Distance matrix methods**</span>

<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MacT">MacT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ODEN">ODEN</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON">TREECON</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RESTSITE">RESTSITE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NTSYSpc">NTSYSpc</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#METREE">METREE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GDA">GDA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lintre">Lintre</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#gmaes">gmaes</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DENDRON">DENDRON</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#BIONJ">BIONJ</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA">TFPGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#sendbs">sendbs</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#nneighbor">nneighbor</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#weighbor">weighbor</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS">DNASIS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MINSPNET">MINSPNET</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#vCEBL">PEBBLE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#qclust">qclust</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP">PTP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree">SplitsTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FastME">FastME</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MacVector">MacVector</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#QuickTree">QuickTree</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist">ProfDist</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START">START2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STC">STC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer">CBCAnalyzer</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Rate4Site">Rate4Site</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IDEA">IDEA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PC-ORD">PC-ORD</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NINJA">NINJA</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STATIO">Statio</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TIMER">TIMER</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Ancestor">Ancestor</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ANC-GENE">ANC-GENE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Bn-Bs">Bn-Bs</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Distances">**Computation of distances**</span>

<table>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RAPDistance">RAPDistance</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MULTICOMP">MULTICOMP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIPLOMO">DIPLOMO</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#OSA">OSA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RESTSITE">RESTSITE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NTSYSpc">NTSYSpc</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GCUA">GCUA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DERANGE2">DERANGE2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#popgene">POPGENE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA">TFPGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#REAP">REAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RSTCALC">RSTCALC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#genetix">Genetix</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTANCE">DISTANCE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#sendbs">sendbs</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DnaSP">DnaSP</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#puzzleboot">puzzleboot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#qclust">qclust</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FSTAT">FSTAT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Phyltools">Phyltools</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA">YCDMA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NSA">NSA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#LDDist">LDDist</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DIVAGE">DIVAGE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Genepop">Genepop</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START">START2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Swaap">Swaap</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SwaapPH">Swaap PH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SPAGeDi">SPAGeDi</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer">CBCAnalyzer</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rRNA">rRNA phylogeny</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GenoDive">GenoDive</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#analysis">analysis</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTREE">DISTREE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Bn-Bs">Bn-Bs</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#HON-new">HON-new</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Likelihood">**Maximum likelihood methods**</span>

<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAml">fastDNAml</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MOLPHY">MOLPHY</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectrum">Spectrum</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree">SplitsTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PASSML">PASSML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">Modeltest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#dnarates">dnarates</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DT-ModSel">DT-ModSel</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAmlRev">fastDNAmlRev</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RevDNArates">RevDNArates</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rate-evolution">rate-evolution</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CONSEL">CONSEL</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO">PLATO</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP">PTP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s-bootstrap">r8s-bootstrap</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BootPHYML">BootPHYML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARBOOT">PARBOOT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#p4">p4</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rhino">Rhino</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate">TipDate</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrAIC">MrAIC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modelfit">Modelfit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARAT">PARAT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIFRITZ">ALIFRITZ</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyNav">PhyNav</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DPRML">DPRML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MultiPhyl">MultiPhyl</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SSA">SSA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CoMET">CoMET</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mac5">Mac5</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYSIG">PHYSIG</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Rate4Site">Rate4Site</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#aLRT">aLRT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#McRate">McRate</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EREM">EREM</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PROCOV">PROCOV</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloCoCo">PhyloCoCo</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#SeqState">SeqState</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Leaphy">Leaphy</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NHML">NHML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SLR">SLR</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rRNA">rRNA phylogeny</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Concaterpillar">Concaterpillar</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#NEPAL">NEPAL</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodeAxe">CodeAxe</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FastTree">FastTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#nhPhyML">nhPhyML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyML-Multi">PhyML-Multi</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#raxmlGUI">raxmlGUI</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree">MixtureTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GZ-Gamma">GZ-Gamma</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Bayesian">**Bayesian inference methods**</span>

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAMBE">BAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesTreeScan">MrBayes tree scanners</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#p4">p4</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesPlugin">MrBayesPlugin</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloBayes">PhyloBayes</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Cadence">Cadence</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Multidivtime">Multidivtime</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST">BEST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#AMBIORE">AMBIORE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#bms_runner">bms_runner</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#burntrees">burntrees</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ANC-GENE">ANC-GENE</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Quartets">**Quartets methods**</span>

  - [TREE-PUZZLE](https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE)
  - [SplitsTree](https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree)
  - [PHYLTEST](https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST)
  - [GEOMETRY](https://evolution.gs.washington.edu/phylip/software.etc1.html#GEOMETRY)
  - [PICA](https://evolution.gs.washington.edu/phylip/software.etc2.html#PICA)
  - [Darwin](https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin)
  - [PhyloQuart](https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloQuart)
  - [Willson quartets
    programs](https://evolution.gs.washington.edu/phylip/software.etc1.html#Willson)
  - [Gambit](https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit)
  - [IQPNNI](https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI)
  - [STC](https://evolution.gs.washington.edu/phylip/software.dist.html#STC)
  - [Quartet
    Suite](https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite)
  - [LEVEL2](https://evolution.gs.washington.edu/phylip/software.etc1.html#LEVEL2)
  - [Bosque](https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque)

<span id="AI">**Artificial-intelligence and genetic algorithms
methods**</span>

  - [PTP](https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP)
  - [GAPars](https://evolution.gs.washington.edu/phylip/software.pars.html#GAPars)
  - [MetaPIGA](https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA)
  - [GARLI](https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI)

<span id="Invariants">**Invariants (or Evolutionary Parsimony)
methods**</span>

  - [PHYLIP](https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP)
  - [PAUP\*](https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP)
  - [PaupUp](https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp)
  - [BIRCH](https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH)
  - [EMBOSS](https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS)

<span id="Interactive">**Interactive tree manipulation**</span>

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PDAP">PDAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeTool">TreeTool</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#WINCLADA">WINCLADA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeEdit">TreeEdit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeExplorer">TreeExplorer</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeThief">TreeThief</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon">RadCon</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#mavric">Mavric</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treeme">TreeMe</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw">ArboDraw</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung">Notung</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker">TreeMaker</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SimpleClade">SimpleClade</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Forest">Forest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Recombinant">**Looking for hybridization or recombination
events**</span>

<table>
<colgroup>
<col style="width: 100%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RecPars">RecPars</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#partimatrix">partimatrix</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO">PLATO</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START">START2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Likewind">Likewind</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DualBrothers">DualBrothers</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#cBrother">cBrother</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HGT">HGT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloNet">PhyloNet</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator">EvolSimulator</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Concaterpillar">Concaterpillar</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyML-Multi">PhyML-Multi</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RDP3">RDP3</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Bootstrapping">**Bootstrapping and other measures of
support**</span> (for Bayesian support methods see the [Bayesian
inference
list](https://evolution.gs.washington.edu/phylip/software.html#Bayesian)
above).

<table>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Random%20Cladistics">Random Cladistics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#AutoDecay">AutoDecay</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TreeRot">TreeRot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DNA%20Stacks">DNA Stacks</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#OSA">OSA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lintre">Lintre</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#sog">sog</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARBOOT">PARBOOT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PICA">PICA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">ModelTest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TAXEQ3">TAXEQ3</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAMBE">BAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#puzzleboot">puzzleboot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CodonBootstrap">CodonBootstrap</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CONSEL">CONSEL</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Phyltools">Phyltools</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s-bootstrap">r8s-bootstrap</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesTreeScan">MrBayes tree scanners</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BootPHYML">BootPHYML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute">Permute!</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ELW">ELW</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MultiPhyl">MultiPhyl</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GHOSTS">GHOSTS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#scaleboot">scaleboot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#aLRT">aLRT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloBayes">PhyloBayes</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloNet">PhyloNet</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FastTree">FastTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#burntrees">burntrees</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#raxmlGUI">raxmlGUI</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree">MixtureTree</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Compatibility">**Compatibility analysis**</span>

  - [COMPROB](https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPROB)
  - [PHYLIP](https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP)
  - [PICA](https://evolution.gs.washington.edu/phylip/software.etc2.html#PICA)
  - [partimatrix](https://evolution.gs.washington.edu/phylip/software.etc1.html#partimatrix)
  - [PPH](https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH)
  - [Spectronet](https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet)
  - [BIRCH](https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH)
  - [EMBOSS](https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS)
  - [Murka](https://evolution.gs.washington.edu/phylip/software.pars.html#Murka)
  - [Phybase](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase)
  - [TIGER](https://evolution.gs.washington.edu/phylip/software.etc1.html#TIGER)

<span id="Consensus">**Consensus trees, subtrees, supertrees, and
distances between trees**</span>

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP">TREEMAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NTSYSpc">NTSYSpc</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#REDCON">REDCON</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TAXEQ3">TAXEQ3</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon">RadCon</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RobinsonFoulds">Robinson and Foulds distance</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Clann">Clann</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyNav">PhyNav</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuperTree">SuperTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree_scripts">Supertree scripts</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree">Supertree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HeuristicMRF2">HeuristicMRF2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TopD/fMts">TopD/fMts</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rainbow">Rainbow</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhySIC">PhySIC_IST</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloSort">PhyloSort</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#raxmlGUI">raxmlGUI</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Alignment">**Tree-based sequence alignment**</span>

  - [TreeAlign](https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeAlign)
  - [ClustalW](https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW)
  - [MALIGN](https://evolution.gs.washington.edu/phylip/software.etc1.html#MALIGN)
  - [GeneDoc](https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc)
  - [DAMBE](https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE)
  - [POY](https://evolution.gs.washington.edu/phylip/software.etc1.html#POY)
  - [ALIGN](https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIGN)
  - [DNASIS](https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS)
  - [FootPrinter](https://evolution.gs.washington.edu/phylip/software.pars.html#FootPrinter)
  - [ALIFRITZ](https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIFRITZ)
  - [T-Coffee](https://evolution.gs.washington.edu/phylip/software.etc1.html#T-Coffee)
  - [ArboDraw](https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw)
  - [BAli-Phy](https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy)
  - [Geneious](https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious)
  - [BIRCH](https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH)
  - [MAFFT](https://evolution.gs.washington.edu/phylip/software.etc1.html#MAFFT)
  - [LOBSTER](https://evolution.gs.washington.edu/phylip/software%20etc1.html#SATCHMO)
  - [DART](https://evolution.gs.washington.edu/phylip/software.etc2.html#DART)
  - [MUSCLE](https://evolution.gs.washington.edu/phylip/software.etc1.html#MUSCLE)
  - [Bosque](https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque)
  - [EMBOSS](https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS)
  - [SeaView](https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView)
  - [SuiteMSA](https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA)

<span id="Genomics">**Gene duplication and genomic analysis**</span>

  - [DERANGE2](https://evolution.gs.washington.edu/phylip/software.dist.html#DERANGE2)
  - [FORESTER](https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER)
  - [BPAnalysis](https://evolution.gs.washington.edu/phylip/software.etc1.html#BPAnalysis)
  - [Notung](https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung)
  - [TopD/fMts](https://evolution.gs.washington.edu/phylip/software.etc2.html#TopD/fMts)
  - [DTscore](https://evolution.gs.washington.edu/phylip/software.etc1.html#DTscore)
  - [DART](https://evolution.gs.washington.edu/phylip/software.etc2.html#DART)
  - [gtp](https://evolution.gs.washington.edu/phylip/software.etc1.html#gtp)
  - [EvolSimulator](https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator)
  - [DTdraw](https://evolution.gs.washington.edu/phylip/software.etc2.html#DTdraw)
  - [Mgenome](https://evolution.gs.washington.edu/phylip/software.etc1.html#Mgenome)
  - [Tree
    Tracker](https://evolution.gs.washington.edu/phylip/software.etc1.html#Tree%20Tracker)
  - [bms\_runner](https://evolution.gs.washington.edu/phylip/software.etc2.html#bms_runner)
  - [MANTiS](https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS)
  - [ETE](https://evolution.gs.washington.edu/phylip/software.pars.html#ETE)
  - [MANTiS](https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS)
  - [iGTP](https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP)

<span id="Biogeography">**Biogeographic analysis and host-parasite
comparison**</span>

  - [COMPONENT](https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT)
  - [TREEMAP](https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP)
  - [DIVA](https://evolution.gs.washington.edu/phylip/software.etc1.html#DIVA)
  - [TreeFitter](https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeFitter)
  - [GEODIS](https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS)
  - [Tarzan](https://evolution.gs.washington.edu/phylip/software.etc1.html#Tarzan)
  - [ParaFit](https://evolution.gs.washington.edu/phylip/software.etc1.html#ParaFit)
  - [Phylocom](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom)
  - [AxParafit](https://evolution.gs.washington.edu/phylip/software.etc1.html#AxParafit)
  - [GenGIS](https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS)
  - [S-DIVA](https://evolution.gs.washington.edu/phylip/software.etc1.html#S-DIVA)
  - [Lagrange](https://evolution.gs.washington.edu/phylip/software.etc1.html#Lagrange)
  - [CoRe-PA](https://evolution.gs.washington.edu/phylip/software.etc1.html#CoRe-PA)
  - [Jane](https://evolution.gs.washington.edu/phylip/software.etc1.html#Jane)

<span id="Comparative">**Comparative method analysis**</span>

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC">CAIC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PDAP">PDAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ANCML">ANCML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RIND">RIND</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MacroCAIC">MacroCAIC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PI">Phylogenetic Independence</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute">Permute!</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Parsimov">Parsimov</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIVERGE">DIVERGE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#IDC">IDC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#OUCH">OUCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYSIG">PHYSIG</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#bms_runner">bms_runner</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SLOUCH">SLOUCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Simulation">**Simulation of trees or data**</span>

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Seq-Gen">Seq-Gen</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treevolve">Treevolve and PTreevolve</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PSeq-Gen">PSeq-Gen</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ROSE">ROSE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ProSeq">ProSeq</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen">Phylogen</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SGRunner">SGRunner</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#apTreeshape">apTreeshape</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Simprot">Simprot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EREM">EREM</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#indel-Seq-Gen">indel-Seq-Gen</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DAWG">DAWG</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolveAGene3">EvolveAGene3</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator">EvolSimulator</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon">Recodon</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon">NetRecodon</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a><br />
</li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Shapes">**Examination of shapes of trees**</span>

  - [MacroCAIC](https://evolution.gs.washington.edu/phylip/software.etc2.html#MacroCAIC)
  - [Genie](https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie)
  - [PAL](https://evolution.gs.washington.edu/phylip/software.pars.html#PAL)
  - [Vanilla](https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla)
  - [RadCon](https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon)
  - [BRANCHLENGTH](https://evolution.gs.washington.edu/phylip/software.etc2.html#branchlength)
  - [APE](https://evolution.gs.washington.edu/phylip/software.etc2.html#APE)
  - [Tracer](https://evolution.gs.washington.edu/phylip/software.etc1.html#Tracer)
  - [SymmeTREE](https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE)
  - [TreeScan](https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan)
  - [MESA](https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA)
  - [apTreeshape](https://evolution.gs.washington.edu/phylip/software.etc2.html#apTreeshape)
  - [TreeStat](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat)
  - [CTree](https://evolution.gs.washington.edu/phylip/software.dist.html#CTree)
  - [Phyutility](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility)
  - [laser](https://evolution.gs.washington.edu/phylip/software.etc1.html#laser)
  - [PhyRe](https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyRe)
  - [Bio::Phylo](https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo)

<span id="Stratigraphy">**Clocks, dating and stratigraphy**</span>

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate">QDate</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">Modeltest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RRTree">RRTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#vCEBL">PEBBLE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeEdit">TreeEdit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAL">PAL</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rate-evolution">rate-evolution</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#branchlength">BRANCHLENGTH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s">r8s</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate">TipDate</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rhino">Rhino</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GHOSTS">GHOSTS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Cadence">Cadence</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Multidivtime">Multidivtime</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodonRates">CodonRates</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PATHd8">PATHd8</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#McRate">McRate</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloBayes">PhyloBayes</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GRate">GRate</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NHML">NHML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TIMER">TIMER</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Modelselection">**Model Selection**</span>

  - [Modeltest](https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest)
  - [MrMTgui](https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui)
  - [MrModeltest](https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest)
  - [Porn\*](https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*)
  - [ModelGenerator](https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator)
  - [ProtTest](https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest)
  - [MrAIC](https://evolution.gs.washington.edu/phylip/software.etc1.html#MrAIC)
  - [Modelfit](https://evolution.gs.washington.edu/phylip/software.etc1.html#Modelfit)
  - [DT-ModSel](https://evolution.gs.washington.edu/phylip/software.etc1.html#DT-ModSel)
  - [BayesTraits](https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits)
  - [Kakusan4](https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan)
  - [MAPPS](https://evolution.gs.washington.edu/phylip/software.etc1.html#MAPPS)
  - [DART](https://evolution.gs.washington.edu/phylip/software.etc2.html#DART)
  - [Concaterpillar](https://evolution.gs.washington.edu/phylip/software.etc1.html#Concaterpillar)
  - [Statio](https://evolution.gs.washington.edu/phylip/software.dist.html#STATIO)
  - [jMODELTEST](https://evolution.gs.washington.edu/phylip/software.etc1.html#jMODELTEST)

<span id="Description">**Description or prediction of data from
trees**</span>

  - [CONSERVE](https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE)
  - [TreeDis](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDis)
  - [EDIBLE](https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE)
  - [RIND](https://evolution.gs.washington.edu/phylip/software.etc2.html#RIND)
  - [Jevtrace](https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace)
  - [SIMMAP](https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP)
  - [TreeSAAP](https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP)
  - [DIVERGE](https://evolution.gs.washington.edu/phylip/software.etc2.html#DIVERGE)
  - [MESA](https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA)
  - [FASTML](https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML)

<span id="Plotting">**Tree plotting/drawing**</span>

<table>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeTool">TreeTool</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NJplot">NJplot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DendroMaker">DendroMaker</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDrawDeck">Tree Draw Deck</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron">Phylodendron</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#unrooted">unrooted</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON">TREECON</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#mavric">Mavric</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeExplorer">TreeExplorer</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeThief">TreeThief</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloTreeDraw">Phylogenetic Tree Drawing</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSetViz">TreeSetViz</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw">ArboDraw</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung">Notung</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DigTree">DigTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Paloverde">Paloverde</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MrEnt">MrEnt</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree">FigTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree">HyperTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeoPhyloBuilder">GeoPhyloBuilder</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSnatcher">TreeSnatcher Plus</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DTdraw">DTdraw</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#S-DIVA">S-DIVA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree">MixtureTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Archaeopteryx">Archaeopteryx</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Management">**Sequence management/job submission**</span>

<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Random%20Cladistics">Random Cladistics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GDE">GDE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MUST">MUST 2000</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DNA%20Stacks">DNA Stacks</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARBOOT">PARBOOT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BioEdit">BioEdit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#WebPHYLIP">Singapore PHYLIP web interface</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#W2H">W2H</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyledit">Phyledit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeneStudioPro">GeneStudio Pro</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DPRML">DPRML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MBEToolbox">MBEToolbox</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PISE">PISE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a></li>
</ul></td>
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PyCogent">PyCogent</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYDIT">PHYDIT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a></li>
</ul></td>
</tr>
</tbody>
</table>

<span id="Teaching">**Teaching about phylogenies**</span>

  - [Phylogenetic
    Investigator](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylogenetic%20Investigator)
  - [Phylap](https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylap)
  - [Dnatree](https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree)
  - [SimpleClade](https://evolution.gs.washington.edu/phylip/software.etc2.html#SimpleClade)

-----

<span id="systems"></span>

<table>
<colgroup>
<col style="width: 100%" />
</colgroup>
<tbody>
<tr class="odd">
<td><strong>Table of contents by computer systems<br />
on which they work</strong></td>
</tr>
</tbody>
</table>

### [Unix (source code in C or executables)](https://evolution.gs.washington.edu/phylip/software.html#Unix)

### [Multiplatform interpreters (Java, Perl, Python, R, MATLAB)](https://evolution.gs.washington.edu/phylip/software.html#interpreters)

### 

### [PC's](https://evolution.gs.washington.edu/phylip/software.html#PCs)

### ... [under Windows](https://evolution.gs.washington.edu/phylip/software.html#Windows)

### ... [under DOS or in a Windows Command tool "DOS box"](https://evolution.gs.washington.edu/phylip/software.html#DOS)

### [Macintoshes (under Mac OS X or Mac OS)](https://evolution.gs.washington.edu/phylip/software.html#Macintosh)

### [**e-mail or Web servers** that can analyze data for you](https://evolution.gs.washington.edu/phylip/software.serv.html)

-----

  - <span id="Unix">**Unix (source code in C or executables).**</span> I
    have included programs that are available as C source code because
    most Unix workstations have a C compiler. (Programs in other
    compiled languages such as FORTRAN and Pascal, and in interpreted
    languages such as Java, Perl, Python, or R are also included), as
    are Java executables. For many of these the programs can also be
    compiled or run on Windows or Mac OS X systems if they have the
    appropriate compilers or interpreters loaded.
    
    <table>
    <colgroup>
    <col style="width: 20%" />
    <col style="width: 20%" />
    <col style="width: 20%" />
    <col style="width: 20%" />
    <col style="width: 20%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ODEN">ODEN</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lintre">Lintre</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#OSA">OSA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAml">fastDNAml</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MOLPHY">MOLPHY</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree">SplitsTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeAlign">TreeAlign</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW">ClustalW</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MALIGN">MALIGN</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc">GeneDoc</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Seq-Gen">Seq-Gen</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeTool">TreeTool</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GDE">GDE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#sog">sog</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron">Phylodendron</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treevolve">Treevolve and PTreevolve</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PSeq-Gen">PSeq-Gen</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#gmaes">gmaes</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GCUA">GCUA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DERANGE2">DERANGE2</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#BIONJ">BIONJ</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ANCML">ANCML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate">QDate</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PASSML">PASSML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RecPars">RecPars</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARBOOT">PARBOOT</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ARB">ARB</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTANCE">DISTANCE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Darwin">Darwin</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#sendbs">sendbs</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#partimatrix">partimatrix</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAMBE">BAMBE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#nneighbor">nneighbor</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#unrooted">unrooted</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ROSE">ROSE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#weighbor">weighbor</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloQuart">PhyloQuart</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#puzzleboot">puzzleboot</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Willson">Willson quartets programs</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#POY">POY</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RIND">RIND</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RRTree">RRTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#mavric">Mavric</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#dnarates">dnarates</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie">Genie</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla">Vanilla</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#qclust">qclust</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAmlRev">fastDNAmlRev</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RevDNArates">RevDNArates</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#branchlength">BRANCHLENGTH</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TCS">TCS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CONSEL">CONSEL</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#W2H">W2H</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GAPars">GAPars</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s">r8s</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO">PLATO</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FastME">FastME</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloTreeDraw">Phylogenetic Tree Drawing</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s-bootstrap">r8s-bootstrap</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesTreeScan">MrBayes tree scanners</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RobinsonFoulds">Robinson and Foulds distance</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Clann">Clann</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BootPHYML">BootPHYML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#LDDist">LDDist</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#p4">p4</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Porn*">Porn*</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen">Phylogen</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rhino">Rhino</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate">TipDate</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylap">Phylap</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree">Dnatree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#QuickTree">QuickTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#FootPrinter">FootPrinter</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BPAnalysis">BPAnalysis</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS">GEODIS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSetViz">TreeSetViz</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist">ProfDist</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrAIC">MrAIC</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modelfit">Modelfit</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PARAT">PARAT</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIFRITZ">ALIFRITZ</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyNav">PhyNav</a></li>
    </ul></td>
    </tr>
    <tr class="even">
    <td> </td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
    </tr>
    <tr class="odd">
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STC">STC</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Likewind">Likewind</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ELW">ELW</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree_scripts">Supertree scripts</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Parsimov">Parsimov</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIVERGE">DIVERGE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#T-Coffee">T-Coffee</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer">CBCAnalyzer</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GHOSTS">GHOSTS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Tarzan">Tarzan</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DT-ModSel">DT-ModSel</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DualBrothers">DualBrothers</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#apTreeshape">apTreeshape</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Multidivtime">Multidivtime</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mgenome">Mgenome</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ParaFit">ParaFit</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#IDC">IDC</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker">TreeMaker</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodonRates">CodonRates</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree">Supertree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#OUCH">OUCH</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DigTree">DigTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mac5">Mac5</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Paloverde">Paloverde</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HeuristicMRF2">HeuristicMRF2</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CRANN">CRANN</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PATHd8">PATHd8</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAFFT">MAFFT</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat">TreeStat</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree">FigTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYSIG">PHYSIG</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#scaleboot">scaleboot</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#cBrother">cBrother</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesPlugin">MrBayesPlugin</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SATCHMO">LOBSTER</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Rate4Site">Rate4Site</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TopD/fMts">TopD/fMts</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rainbow">Rainbow</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#McRate">McRate</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree">HyperTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloBayes">PhyloBayes</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Forest">Forest</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhySIC">PhySIC_IST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Simprot">Simprot</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST">BEST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EREM">EREM</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#indel-Seq-Gen">indel-Seq-Gen</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MBEToolbox">MBEToolbox</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DTscore">DTscore</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PROCOV">PROCOV</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DAWG">DAWG</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#LEVEL2">LEVEL2</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PSODA">PSODA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloSort">PhyloSort</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PISE">PISE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MUSCLE">MUSCLE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#AMBIORE">AMBIORE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HGT">HGT</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NHML">NHML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SLR">SLR</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rRNA">rRNA phylogeny</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#AxParafit">AxParafit</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolveAGene3">EvolveAGene3</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#gtp">gtp</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSnatcher">TreeSnatcher Plus</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolSimulator">EvolSimulator</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Concaterpillar">Concaterpillar</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DTdraw">DTdraw</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#NEPAL">NEPAL</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYLLAB">PHYLLAB</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bioinformatics_Toolbox">Bioinformatics_Toolbox</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Tree%20Tracker">Tree Tracker</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#analysis">analysis</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodeAxe">CodeAxe</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyutility">Phyutility</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#phangorn">phangorn</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FastTree">FastTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#laser">laser</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#bms_runner">bms_runner</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#burntrees">burntrees</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SLOUCH">SLOUCH</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS">MANTiS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Freqpars">Freqpars</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE">CONSERVE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
    </ul></td>
    </tr>
    <tr class="even">
    <td> </td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
    </tr>
    <tr class="odd">
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTREE">DISTREE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PyCogent">PyCogent</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DendroPy">DendroPy</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC">CAIC</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NINJA">NINJA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MUST">MUST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#nhPhyML">nhPhyML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyML-Multi">PhyML-Multi</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon">Recodon</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon">NetRecodon</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lagrange">Lagrange</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CoRe-PA">CoRe-PA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MixtureTree">MixtureTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TIGER">TIGER</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Jane">Jane</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GZ-Gamma">GZ-Gamma</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Archaeopteryx">Archaeopteryx</a>
    <ul>
    </ul></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Crux">Crux</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Ancestor">Ancestor</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ANC-GENE">ANC-GENE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Bn-Bs">Bn-Bs</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#HON-new">HON-new</a></li>
    </ul></td>
    <td></td>
    <td></td>
    <td></td>
    </tr>
    </tbody>
    </table>

  - <span id="interpreters">**Multiplatform interpreters (Java, Perl,
    Python, R, MATLAB)**</span>. These are programs that are written in
    a language that is interpreted, step by step, in real time by a
    computing environment that can run on many different operating
    systems.
    
    *(I am just starting to list interpreter code here. Until recently
    it was listed under Unix, Windows and/or Mac OS X). Until I finish
    transferring interpreter code here this list will be incomplete, and
    you will find many programs written in interpreted languages
    there).*
    
    **Java**
    
      - [PAL](https://evolution.gs.washington.edu/phylip/software.pars.html#PAL)
      - [Mesquite](https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite)
      - [BIRCH](https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH)
      - [PRAP](https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP)
      - [SeqState](https://evolution.gs.washington.edu/phylip/software.pars.html#SeqState)
      - [TCS](https://evolution.gs.washington.edu/phylip/software.pars.html#TCS)
      - [IDEA](https://evolution.gs.washington.edu/phylip/software.etc1.html#IDEA)
      - [PhyloNet](https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloNet)
      - [Notung](https://evolution.gs.washington.edu/phylip/software.etc1.html#Notung)
      - [Vanilla](https://evolution.gs.washington.edu/phylip/software.pars.html#Vanilla)
      - [NINJA](https://evolution.gs.washington.edu/phylip/software.dist.html#NINJA)
      - [qclust](https://evolution.gs.washington.edu/phylip/software.dist.html#qclust)
      - [PEBBLE](https://evolution.gs.washington.edu/phylip/software.dist.html#vCEBL)
      - [SplitsTree](https://evolution.gs.washington.edu/phylip/software.etc1.html#SplitsTree)
      - [SeqPup](https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup)
      - [DPRML](https://evolution.gs.washington.edu/phylip/software.etc1.html#DPRML)
      - [MultiPhyl](https://evolution.gs.washington.edu/phylip/software.etc1.html#MultiPhyl)
      - [Treefinder](https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder)
      - [PhyloCoCo](https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloCoCo)
      - [ProtTest](https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest)
      - [CoMET](https://evolution.gs.washington.edu/phylip/software.etc2.html#CoMET)
      - [Segminator](https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator)
      - [jMODELTEST](https://evolution.gs.washington.edu/phylip/software.etc1.html#jMODELTEST)

  - <span id="PCs">**PC's**</span>
    
      - <span id="Windows">**as Windows executables**</span> (not
        counting executing in a "DOS box"). Programs available as source
        code which is Windows-specific are listed here. Java executables
        are also included. (Note that compilers available on Windows
        systems, particularly the free Cygwin and MinGW compilers, can
        also be used to compile many of the programs listed above under
        Unix generic source code). Programs run in interpreted
        environments such as Perl, Python, R or MATLAB can also be run
        under Windows if the proper environment is installed. These
        programs are listed above under Unix.
        <table>
        <colgroup>
        <col style="width: 20%" />
        <col style="width: 20%" />
        <col style="width: 20%" />
        <col style="width: 20%" />
        <col style="width: 20%" />
        </colgroup>
        <tbody>
        <tr class="odd">
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON">TREECON</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GDA">GDA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MOLPHY">MOLPHY</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc">GeneDoc</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP">TREEMAP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RAPDistance">RAPDistance</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron">Phylodendron</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#popgene">POPGENE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA">TFPGA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GeneTree">GeneTree</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RSTCALC">RSTCALC</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#genetix">Genetix</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NJplot">NJplot</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#unrooted">unrooted</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DnaSP">DnaSP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS">DNASIS</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MINSPNET">MINSPNET</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BioEdit">BioEdit</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ProSeq">ProSeq</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#WINCLADA">WINCLADA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#NONA">NONA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PI">Phylogenetic Independence</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeExplorer">TreeExplorer</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie">Genie</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EDIBLE">EDIBLE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s">r8s</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyledit">Phyledit</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PTP">PTP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DIVA">DIVA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeFitter">TreeFitter</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PAST">PAST</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeneStudioPro">GeneStudio Pro</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Phyltools">Phyltools</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mgenome">Mgenome</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHASE">PHASE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA">YCDMA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NSA">NSA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Clann">Clann</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrMTgui">MrMTgui</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectronet">Spectronet</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen">Phylogen</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylap">Phylap</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree">Dnatree</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS">GEODIS</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSetViz">TreeSetViz</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treeme">TreeMe</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#simplot">Simplot</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist">ProfDist</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#START">START2</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STC">STC</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Swaap">Swaap</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SwaapPH">Swaap PH</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIVERGE">DIVERGE</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NimbleTree">NimbleTree</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ArboDraw">ArboDraw</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SPAGeDi">SPAGeDi</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CBCAnalyzer">CBCAnalyzer</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DualBrothers">DualBrothers</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PaupUp">PaupUp</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SSA">SSA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Multidivtime">Multidivtime</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ParaFit">ParaFit</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#IDC">IDC</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker">TreeMaker</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CodonRates">CodonRates</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DigTree">DigTree</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mac5">Mac5</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MrEnt">MrEnt</a></li>
        </ul></td>
        </tr>
        <tr class="even">
        <td> </td>
        <td></td>
        <td></td>
        <td></td>
        <td></td>
        </tr>
        <tr class="odd">
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SimpleClade">SimpleClade</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CRANN">CRANN</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PATHd8">PATHd8</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAFFT">MAFFT</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat">TreeStat</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree">FigTree</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesPlugin">MrBayesPlugin</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SATCHMO">LOBSTER</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FASTML">FASTML</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Rate4Site">Rate4Site</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rainbow">Rainbow</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#aLRT">aLRT</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#McRate">McRate</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree">HyperTree</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GeoPhyloBuilder">GeoPhyloBuilder</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Simprot">Simprot</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST">BEST</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EREM">EREM</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MBEToolbox">MBEToolbox</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DTscore">DTscore</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#LEVEL2">LEVEL2</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PSODA">PSODA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloSort">PhyloSort</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MUSCLE">MUSCLE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Leaphy">Leaphy</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GRate">GRate</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SLR">SLR</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#rRNA">rRNA phylogeny</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#AxParafit">AxParafit</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DTscore">DTscore</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PROCOV">PROCOV</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DART">DART</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#EEEP">EEEP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE">CONSERVE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DAWG">DAWG</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolveAGene3">EvolveAGene3</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSnatcher">TreeSnatcher Plus</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#FastTree">FastTree</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Murka">Murka</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS">MANTiS</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PC-ORD">PC-ORD</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#S-DIVA">S-DIVA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYDIT">PHYDIT</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RDP3">RDP3</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyRe">PhyRe</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon">Recodon</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon">NetRecodon</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CoRe-PA">CoRe-PA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TIGER">TIGER</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Jane">Jane</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TIMER">TIMER</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#HON-new">HON-new</a></li>
        </ul></td>
        <td></td>
        </tr>
        </tbody>
        </table>
      - <span id="DOS">**under DOS**</span> (MSDOS, PCDOS) or in a
        Command Prompt accessory "DOS box" in Windows
        <table>
        <colgroup>
        <col style="width: 33%" />
        <col style="width: 33%" />
        <col style="width: 33%" />
        </colgroup>
        <tbody>
        <tr class="odd">
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Hennig86">Hennig86</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MEGA">MEGA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#RA">RA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#NoNA">NONA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TREECON">TREECON</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RESTSITE">RESTSITE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#NTSYSpc">NTSYSpc</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#METREE">METREE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PHYLTEST">PHYLTEST</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RAPDistance">RAPDistance</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DIPLOMO">DIPLOMO</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW">ClustalW</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MALIGN">MALIGN</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GeneDoc">GeneDoc</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Random%20Cladistics">Random Cladistics</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GEOMETRY">GEOMETRY</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PDAP">PDAP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PICA">PICA</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#REDCON">REDCON</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TAXEQ3">TAXEQ3</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#BIONJ">BIONJ</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ANCML">ANCML</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#REAP">REAP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lintre">Lintre</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#sendbs">sendbs</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#weighbor">weighbor</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#POY">POY</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDis">TreeDis</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Gambit">Gambit</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CONSEL">CONSEL</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FSTAT">FSTAT</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FastME">FastME</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate">QDate</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RobinsonFoulds">Robinson and Foulds distance</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DIVAGE">DIVAGE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BPAnalysis">BPAnalysis</a></li>
        </ul></td>
        <td><ul>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Genepop">Genepop</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISTREE">DISTREE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#GZ-Gamma">GZ-Gamma</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#STATIO">Statio</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Ancestor">Ancestor</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ANC-GENE">ANC-GENE</a></li>
        <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Bn-Bs">Bn-Bs</a></li>
        </ul></td>
        </tr>
        </tbody>
        </table>

  - <span id="Macintosh">**Macintosh Mac OS X or Mac OS
    executables**</span>. Java executables are also included. (Note that
    most of the programs in the Unix/Linux list above can also be
    compiled for Mac OS X using the GCC compiler that is distributed
    with Mac OS X). Programs run in the interpreted environments Perl
    and Python can also be run in Mac OS X as these are included in the
    Mac OS X distribution. We have not yet included these in the list
    below but should. R or MATLAB can also be run under Mac OS X if the
    proper environment is installed. These programs are listed above
    under Unix.
    
    <table style="width:100%;">
    <colgroup>
    <col style="width: 16%" />
    <col style="width: 16%" />
    <col style="width: 16%" />
    <col style="width: 16%" />
    <col style="width: 16%" />
    <col style="width: 16%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUP">PAUP*</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MacT">MacT</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SeqPup">SeqPup</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREE-PUZZLE">TREE-PUZZLE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#fastDNAml">fastDNAml</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Spectrum">Spectrum</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#AutoDecay">AutoDecay</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CAFCA">CAFCA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ClustalW">ClustalW</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TREEMAP">TREEMAP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC">CAIC</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Seq-Gen">Seq-Gen</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CONSERVE">CONSERVE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeView">TreeView</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NJplot">NJplot</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#DendroMaker">DendroMaker</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MUST">MUST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DNA%20Stacks">DNA Stacks</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylogenetic%20Investigator">Phylogenetic Investigator</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDrawDeck">Tree Draw Deck</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylodendron">Phylodendron</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TreeRot">TreeRot</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Treevolve">Treevolve and PTreevolve</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PSeq-Gen">PSeq-Gen</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#BIONJ">BIONJ</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GCUA">GCUA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GeneTree">GeneTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#QDate">QDate</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#T-REX">T-REX</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#unrooted">unrooted</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#COMPONENT">COMPONENT Lite</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#weighbor">weighbor</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Modeltest">Modeltest</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PAML">PAML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Willson">Willson quartets programs</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ALIGN">ALIGN</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CodonBootstrap">CodonBootstrap</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DNASIS">DNASIS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PLATO">PLATO</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MacroCAIC">MacroCAIC</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RadCon">RadCon</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeEdit">TreeEdit</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#HY-PHY">HY-PHY</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeThief">TreeThief</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Genie">Genie</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayes">MrBayes</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FORESTER">FORESTER</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#r8s">r8s</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GDA">GDA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#syntax">SYN-TAX</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DIVA">DIVA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TreeFitter">TreeFitter</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Phylo_win">Phylo_win</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#treefinder">Treefinder</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PPH">PPH</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MetaPIGA">MetaPIGA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PHYML">PHYML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEAST">BEAST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RobinsonFoulds">Robinson and Foulds distance</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Clann">Clann</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Jevtrace">Jevtrace</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrModeltest">MrModeltest</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BootPHYML">BootPHYML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SymmeTREE">SymmeTREE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeJuxtaposer">TreeJuxtaposer</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MacVector">MacVector</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phyl-O-Gen">Phylogen</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rhino">Rhino</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TipDate">TipDate</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylap">Phylap</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dnatree">Dnatree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ProtTest">ProtTest</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#GEODIS">GEODIS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSetViz">TreeSetViz</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ModelGenerator">ModelGenerator</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#ProfDist">ProfDist</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IQPNNI">IQPNNI</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSAAP">TreeSAAP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute">Permute!</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MESA">MESA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SGRunner">SGRunner</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#DualBrothers">DualBrothers</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Cadence">Cadence</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#ParaFit">ParaFit</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeMaker">TreeMaker</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BAli-Phy">BAli-Phy</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Supertree">Supertree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeDyn">TreeDyn</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Geneious">Geneious</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Mac5">Mac5</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BayesPhylogenies">BayesPhylogenies</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Paloverde">Paloverde</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#CRANN">CRANN</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAPPS">MAPPS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PATHd8">PATHd8</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MAFFT">MAFFT</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GARLI">GARLI</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeStat">TreeStat</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#FigTree">FigTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MrBayesPlugin">MrBayesPlugin</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SEMPHY">SEMPHY</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Quartet_Suite">Quartet Suite</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Rainbow">Rainbow</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#HyperTree">HyperTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Kakusan">Kakusan4</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SWORDS">SWORDS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Dendroscope">Dendroscope</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhySIC">PhySIC_IST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TOPALi">TOPALi</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#BEST">BEST</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#indel-Seq-Gen">indel-Seq-Gen</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#PhyloCoCo">PhyloCoCo</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#LEVEL2">LEVEL2</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PSODA">PSODA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloSort">PhyloSort</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#RAXML">RAxML</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MUSCLE">MUSCLE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#CTree">CTree</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PRAP">PRAP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SLR">SLR</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#AxParafit">AxParafit</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#EvolveAGene3">EvolveAGene3</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeToy">TreeToy</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeSnatcher">TreeSnatcher Plus</a></li>
    </ul></td>
    </tr>
    <tr class="even">
    <td> </td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
    </tr>
    <tr class="odd">
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GAME">GAME</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GenoDive">GenoDive</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PhyloWidget">PhyloWidget</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#tracer">tracer</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#MANTiS">MANTiS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#GenGIS">GenGIS</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio++">Bio++</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#UGENE">UGENE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phybase">Phybase</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#ETE">ETE</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PyCogent">PyCogent</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DendroPy">DendroPy</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#NEPAL">NEPAL</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Bosque">Bosque</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#TreeGraph">TreeGraph 2</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Segminator">Segminator</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#iGTP">iGTP</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bio::Phylo">Bio::Phylo</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Recodon">Recodon</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Lagrange">Lagrange</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#NetRecodon">NetRecodon</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#CoRe-PA">CoRe-PA</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#TIGER">TIGER</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SeaView">SeaView</a></li>
    </ul></td>
    <td><ul>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Jane">Jane</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PAUPRat">PAUPRat</a></li>
    <li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SuiteMSA">SuiteMSA</a></li>
    </ul></td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
    </tr>
    </tbody>
    </table>

-----

<span id="datatypes"></span>

|                                        |
| -------------------------------------- |
| **Analyzing particular types of data** |

Here you will find lists of programs that analyze types of data other
than molecular sequence data. We will gradually expand this list of data
types.

<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
</colgroup>
<tbody>
<tr class="odd">
<td><strong>Microsatellite data</strong>    
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RSTCALC">RSTCALC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Microsat">Microsat</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Populations">Populations</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#MSA">MSA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA">YCDMA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#IM">IMa2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a></li>
</ul></td>
<td><strong>RAPDs, RFLPs, or AFLPs</strong>    
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TFPGA">tfpga</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#RAPDistance">RAPDistance</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GelCompar">GelCompar II</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Bionumerics">Bionumerics</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Winboot">Winboot</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#REAP">REAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#RESTSITE">RESTSITE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#MVSP">MVSP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DENDRON">DENDRON</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Phyltools">Phyltools</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#Network">Network</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BIRCH">BIRCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FAMD">FAMD</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
</ul></td>
<td><strong>Continuous quantitative</strong>    <br />
<strong>characters</strong>
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#MacClade">MacClade</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#mesquite">Mesquite</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#ANCML">ANCML</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#COMPARE">COMPARE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PDAP">PDAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PI">Phylogenetic Independence</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CAIC">CAIC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#treescan">TreeScan</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYLOGR">PHYLOGR</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#IDC">IDC</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#CoMET">CoMET</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#OUCH">OUCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Brownie">Brownie</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#BayesTraits">BayesTraits</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#TNT">TNT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#PHYSIG">PHYSIG</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Cactus-Pie">Cactus-Pie</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#Phylocom">Phylocom</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#pcca">pcca</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Permute">Permute</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc1.html#SIMMAP">SIMMAP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#SLOUCH">SLOUCH</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#PC-ORD">PC-ORD</a></li>
</ul></td>
<td><strong>Gene frequencies</strong><br />
(aside from microsatellite loci)
<ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#PHYLIP">PHYLIP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#DAMBE">DAMBE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#Freqpars">Freqpars</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DISPAN">DISPAN</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GDA">GDA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#popgene">POPGENE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#YCDMA">YCDMA</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#FSTAT">FSTAT</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Arlequin">Arlequin</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DnaSP">DnaSP</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.etc2.html#APE">APE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#DIVAGE">DIVAGE</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#poptrfdos">POPTREE2</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#Genepop">Genepop</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#SPAGeDi">SPAGeDi</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#GenoDive">GenoDive</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#TreeFit">TreeFit</a></li>
<li><a href="https://evolution.gs.washington.edu/phylip/software.dist.html#EMBOSS">EMBOSS</a></li>
</ul></td>
</tr>
</tbody>
</table>

(under construction: more coming soon)

-----

<span id="recent"></span>

|                     |
| ------------------- |
| **Recent listings** |

<span id="recent">**Here are the packages that have most recently been
added to these listings:** (the most recent ones first). Entries are
retained in this list for 6 months. Note also</span>
[below](https://evolution.gs.washington.edu/phylip/software.html#new)
the "waiting list" area listing programs that are to be added. You can
use the submission form
[here](http://evolution.gs.washington.edu/cgi-bin/software-form-new1.pl)
to submit new entries.

<table>
<colgroup>
<col style="width: 100%" />
</colgroup>
<tbody>
<tr class="odd">
<td><ul>
<li>(List is currently empty because I have been unable to do much updating owing to other pressures so no new packages have been added in the last year).</li>
</ul></td>
</tr>
</tbody>
</table>

-----

<span id="changes"></span>

|                    |
| ------------------ |
| **Recent changes** |

**Here are the packages whose entries have most recently been changed:**
The date on which each change was entered is shown. Entries are retained
in this list for 6 months. (Note that changes may be as small as updated
version numbers or a modified web address). The most recent changes are
first.

<table>
<colgroup>
<col style="width: 100%" />
</colgroup>
<tbody>
<tr class="odd">
<td></td>
</tr>
<tr class="even">
<td><ul>
<li><a href="https://evolution.gs.washington.edu/phylip/software.pars.html#LVB">LVB</a> (19 June 2014)</li>
</ul></td>
</tr>
</tbody>
</table>

-----

<span id="otherlists"></span>

|                                       |
| ------------------------------------- |
| **Other lists of phylogeny software** |

  - <span id="otherlists">There is one phylogeny software list even more
    complete and up-to-date than this one: a more recent version of this
    list. If you are reading this on the web pages at our server
    `evolution.gs.washington.edu`, you are reading the most up-to-date
    version. But if you are reading a version stored anywhere else, you
    might want to</span> [look
    here](http://evolution.gs.washington.edu/phylip/software.html)
    instead.
  - Sergios-Orestis Kolokotronis has posted [an extensive
    table](http://research.amnh.org/users/koloko/softlinks/phylogeny.html)
    of phylogeny programs at his site at the American Museum of Natural
    History, and near it are others under headings such as "molecular
    evolution" and "alignment".
  - Wikipedia has a good list of sequence alignment software (including
    both tree-based and non-tree-based alignment methods)
    [here](http://en.wikipedia.org/wiki/List_of_sequence_alignment_software)
    at
    `http://en.wikipedia.org/wiki/List_of_sequence_alignment_software`
  - David Robertson of the Bioinformatics Education and Research at the
    University of Manchester, England, maintains a very informative web
    site at listing programs and their web sites that test for the
    presence of recombination or hybridization events in DNA sequence
    data. It lists some programs that are covered here, and others that
    are outside the scope of these web pages. [That
    site](http://bioinf.man.ac.uk/robertson/recombination/programs.shtml)
    is located at
    `http://bioinf.man.ac.uk/robertson/recombination/programs.shtml`.
  - Mike Robeson at the University of Colorado maintains a page with
    multiple programs listed as [Bioinformatics software for Mac OS
    X](http://rintintin.colorado.edu/~robeson/Bioinformatics/OS%20X%20Software%20%26%20Links.html).
  - The Bioinformatics Organization, a nonprofit group in Hudson,
    Massachusetts, has posted the
    [**bioinformatics.org**](http://www.bioinformatics.org/) web pages.
    These offer a free membership and host open source software projects
    in bioinformatics. They also have a [Molecular
    Linux](http://bioinformatics.org/software/mol_linux.php3) listing of
    Linux programs to carry out bioinformatics tasks, which can be
    sorted by keywords.
  - On Wikipedia there is a [List of phylogenetic tree visualization
    software](http://en.wikipedia.org/wiki/List_of_phylogenetic_tree_visualization_software)
    at
    `http://en.wikipedia.org/wiki/List_of_phylogenetic_tree_visualization_software`
  - The University of California Museum of Paleontology page of
    [Phylogenetics Software
    Resources](http://www.ucmp.berkeley.edu/subway/phylo/phylosoft.html)
    at `http://www.ucmp.berkeley.edu/subway/phylo/phylosoft.html`. A few
    programs are listed, but there is a very nice list of software lists
    there.
  - Andrea Ramge, of Biomax Informatics AG, Martinsried, Germany has
    created the **`bioinformatik.de`** index of resources. It includes a
    [list of
    software](http://www.bioinformatik.de/cgi-bin/browse/Catalog/Software/)
    located at
    `http://www.bioinformatik.de/cgi-bin/browse/Catalog/Software`. The
    phylogeny programs listings there are located within the categories
    for different operating systems. The phylogeny software is under
    "Phylogenetic Analysis" within each operating system.
  - Richard Christen at the Université de Nice, France, has a list of
    [Tree and Tree-software for visualisation and manipulations dealing
    with phylogenetic
    trees](http://bioinfo.unice.fr/biodiv/Tree_editors.html) at
    `http://bioinfo.unice.fr/biodiv/Tree_editors.html`
  - Silvio Nihei at the University of São Paulo in Brazil has produced
    [a list: Programas para
    Filogenia](http://www.ib.usp.br/~silvionihei/programas.htm) in
    Portugese. It concentrates on a small number of programs that mostly
    use parsimony methods.

Don Gilbert, of the Department of Biology of the University of Indiana,
has a good web page on [Free Software in Molecular Biology for Macintosh
and MS Windows
computers](http://iubio.bio.indiana.edu/soft/molbio/Listings.html) as of
1998 at `http://iubio.bio.indiana.edu/soft/molbio/Listings.html`. It
lists some popular packages and all packages and programs kept at the
IUBio ftp server at that time (see [our description of that
server](https://evolution.gs.washington.edu/phylip/software.serv.html#ftpservers)).
Unfortunately the web links on that page are not active so the addresses
must be copied out and pasted into the URL box of your browser.

Genamics, a company located in Hamilton, New Zealand, maintains the
[SoftwareSeek searchable index of bioinformatics
software](http://genamics.com/software/index.htm) at
`http://genamics.com/software/index.htm` in a number of categories. One
of them is Phylogenetic Analysis. They have a reasonably large number of
entries under that heading, though it also includes some statistical
genetics software that is really not phylogenetic. Their listing has
links to the web sites of the software; for those programs that are not
available by Web they maintain copies for download at their server.
Unfortunately, the links to the software are very out of date and few of
them work.

-----

<span id="new"></span>

|                                      |
| ------------------------------------ |
| **New programs waiting to be added** |

<span id="new">This is a "waiting list" showing links to the web pages
of many new phylogeny programs, which I have not yet had time to add to
the main listing. They will be listed there, with a single web link and
no detailed explanation. I hope that this list will gradually shrink as
the new programs are put into the main listing. You can use the
submission form</span>
[here](http://evolution.gs.washington.edu/cgi-bin/software-form.pl) to
submit new entries.

**These are waiting to be added:**

This bunch are all old programs from Masatoshi Nei's lab, available from
the same web page there

  - [ADAPTSITE](http://homes.bio.psu.edu/people/faculty/nei/software.htm),
    intended to estimate positive and negative selection at a single
    amino acid site.

These are not from the Nei site:

  - [PhyRe](http://www.mozoolab.net/index.php/software-download.html)
    infers adequacy of taxon sampling for phylogenetic studies.
  - [phylobase](http://cran.r-project.org/web/packages/phylobase/index.html)
    is an R package that contains a class of functions for comparative
    methods, incorporating one or more trees and trait data.
  - [PhyML-mixtures](http://www.atgc-montpellier.fr/models/index.php?model=mixture),
    a PhyML version for mixture of amino acid models (EX2, EX3, EHO,
    UL2, and UL3).
  - [PhyD\*](http://www.atgc-montpellier.fr/phyd/), Fast NJ-like
    algorithms to deal with incomplete distance matrices.
  - [SDM](http://www.atgc-montpellier.fr/sdm/) a fast distance-based
    approach for tree and supertree building in phylogenomics.
  - [SSIMUL](http://www.atgc-montpellier.fr/ssimul/) does speciation
    signal extraction from multigene families.
  - [Clearcut](http://bioinformatics.hungry.com/clearcut/) carries out
    Relaxed Neighbor Joining (RNJ), a faster NJ-like distance method.
  - [MP-EST](http://code.google.com/p/mp-est/) (also described
    [here](http://cars.desu.edu/faculty/lliu/research/mpest.html)) uses
    trees from different loci to infer a species tree by a
    pseudo-maximum-likelihood method.
  - [TreeRogue](https://stat.ethz.ch/pipermail/r-sig-phylo/2010-October/000816.html),
    an R script for getting trees from published figures of them.
  - [Serial NetEvolve](http://biorg.cis.fiu.edu/SNE/) simulation program
    evolves serially-sampled sequences with or without recombination.
  - [Rococo](http://bibiserv.techfak.uni-bielefeld.de/rococo/)
    reconstructs ancestral gene clusters for a multigene family on a
    given tree.

I will be adding new programs as they come in, and removing programs
from this waiting list as they are added to the main listing. You can
speed up the listing of a program and help me by using the submission
form
[here](http://evolution.gs.washington.edu/cgi-bin/software-form-new1.pl)
to submit entries. Any entries submitted that way (including ones that
are already in the Waiting List) will "jump the queue" and get highest
priority for being added to the full listing.

-----

<span id="news"></span>

|          |
| -------- |
| **News** |

  - (27 May 2012) Adding one entry from the Waiting List each week, have
    caught up with submissions submitted through the web form, and am
    populating a new column in the cross-referenced table, one for
    multiplatform interpreted code such as Java, Perl, Python, R, and
    MatLab. The Java programs are entered in that column. After this I
    will put the others in it, then go back to the main Phylogeny
    Programs front web page and put in a software category for these
    interpreters. If you have a new program, or an old one that I don't
    list, don't wait for me to find it by myself -- I still don't have
    time to, so use the web submission form.
  - (19 March 2012) Well, I should have known that another quarter of
    teaching lay ahead. Now that is done and I should make gradual
    progress over the next two quarters.
  - (23 December 2011) Once again I got stalled by heavy teaching. Now
    resuming again and hope to gradually catch up. One puzzle is how to
    handle R packages. There are a great many of them, and most are not
    listed here yet. It simply is too much work for me to track them all
    down, figure out their features, and make an entry for each one. So
    I will put in only those whose authors use our submission form to
    help create the entry.
  - (20 August 2011) I have resumed adding new submissions that people
    sent in using our software submission forms -- there are about 10
    waiting, some having waited for over 6 months. Apologies for that, I
    was busy teaching and desperately trying to write up old results. I
    hope to gradually add all 10 over the next month or so, one at a
    time.
  - (17 December 2010) I have started (on this 107th anniversary of the
    Wright Brothers' famous first flight) this News section of the page.
    The current status is that I have completed (over the past 2-3
    years) a complete pass through the listings, updating them. However
    of course some may have become outdated since then. Ahead is adding
    new entries, of which I expect there to be 30-40. I am caught up on
    entries that were submitted by the web submission form.
  - (19 December 2010) I have finished adding the ones that were already
    in our Waiting List. Now to take some of the approximately 40 leads
    that I have accumulated and put entries for the relevant ones into
    the Waiting List.
  - (19 June 2014) Things have been stalled for some years but now I am
    gradually (and slowly) resuming adding and correcting entries.
    Please keep using the software submission form, and please be
    patient. I was stalled owing to needing to write grants, and owing
    to not getting them so I now have no programming assistance.
    Progress to being current will be slow. If you are impatient, how
    about volunteering to help? We could set up some web site accessible
    to a team.

|                                     |
| ----------------------------------- |
| **Mysteries you can help us solve** |

**These haven't been entered** because I am not sure what they do and
whether it is covered by this list. *Anyone with relevant information
should let me know.*

  - <span id="news"></span>
    [3item](http://groups.yahoo.com/group/SB1981/files/3item/) extracts
    3-item statements from "areagrams", whatever that means, but can
    only be accessed by joining their Yahoo group. So I don't really
    know what it does.
  - [Dependency
    v2.1](http://www.uhnresearch.ca/labs/tillier/software.htm) uses
    Multiple Interdependency to detect functional interactions between
    amino acids in proteins. Does not seem to actually use a phylogeny.
  - [Codep](http://www.uhnresearch.ca/labs/tillier/software.htm)
    Maximizes co-evolutionary interdependencies to discover interacting
    proteins. Also does not seem to actually use a phylogeny.
  - [PhyloGrapher](http://www.atgc.org/PhyloGrapher/Graph_Ath_NBS.html)
    shows clustering relationships between genes in a genome based on a
    distance matrix. But is it a phylogeny program?
  - [Phylosopher](http://www.genedata.com/products/phylosopher/index_eng.html)
    commercial package for functional genomics said to include some
    phylogeny functions. Does it?
  - [Phylogenator](http://www.multimedialibrary.com/Articles/Jack/Phylogenator.asp)
    server displaying aligned sequences -- does it actually use or
    construct phylogenies?
  - [MultiLocus](http://www.agapow.net/software/multilocus/) calculates
    multiple-locus measures of population differentiation from
    population genetic data. But unless someone can show me that it can
    calculate a measure of distance between two populations within a
    data set, it does not seem appropriate for this list.
  - [CIPRES-KEPLER](http://www.phylo.org/sub_sections/software/)
    Java-based framework for organizing workflow and submitting jobs. I
    am not sure whether any specifically phylogeny-based pieces have yet
    been supplied with this.

      Until I get more information these programs cannot be further
listed.

-----

[To first page of main software
listing](https://evolution.gs.washington.edu/phylip/software.pars.html)

-----

[![\[Phylip icon here\]](./Phylogeny%20Programs_files/PHYLIP.gif) ... to
the PHYLIP home page](https://evolution.gs.washington.edu/phylip.html)
<span id="bottom">  
</span>

<div>

Notices added in compliance with University of Washington requirements
for web sites hosted at the University:     
[Privacy](http://www.washington.edu/online/privacy)    
[Terms](http://www.washington.edu/online/terms)

</div>
